<BASE HREF="http://post.queensu.ca/~forsdyke/bioinfo2.htm">
<html>

<head>
<title>Chargaff's Legacy</title>
<meta name="description" content="Chargaff's four rules on DNA base composition are now 
shown to be fundamental to the understanding of the structure and function of DNA and
the transmission of two levels of genetic information from generation to generation">
<meta name="keywords" content="Base composition, Chargaff's rules, Double-stranded RNA,
 Self/Not-self discrimination, Speciation, Szybalski's transcription direction rule,
 Thermophiles, Erwin Chargaff, Winston Salser, Andrew Ball, Jun-ichi Tomizawa, Naboru Sueoka, 
 Richard Grantham, Herman J. Muller, William Bateson, George Romanes, Francis Crick, 
 Ernst Mayr, G. Ledyard Stebbings, Richard Dawkins">
<meta name="author" content="Donald Forsdyke">
<meta name="publisher" content="Queen's University, Kingston">
</head>

<body>
<div align="center"><center>

<table border="0" cellpadding="15" cellspacing="1" width="100%">
  <tr>
    <td width="50%"><b><font SIZE="5">&nbsp;<p ALIGN="CENTER"></font><a name="Chargaff&#146;s legacy"><font size="7" color="#800040">Chargaff&#146;s legacy</font></a></b></p>
    <p ALIGN="CENTER">&nbsp;</p>
    <b><p ALIGN="CENTER"></b><i>&nbsp;</p>
    <p ALIGN="CENTER"><b><font color="#800040">Reviewed by</font></b></p>
    <p ALIGN="CENTER"><b>Donald R. Forsdyke &amp; James R. Mortimer</b></i></td>
    <td width="50%"><img border="0" src="images/Chargaf2.jpg" alt="Erwin Chargaff" width="377" height="508"></td>
  </tr>
</table>
</center></div><i>

<p ALIGN="CENTER"></i><font SIZE="2">Presented at the Workshop:&nbsp; <font color="#800040"><strong><i>Neutralism and Selectionism. The End of a Debate</i></strong></font>.
Ischia May 3<sup>rd</sup>-6<sup>th</sup> 2000</font><b></p>

<p ALIGN="CENTER"><font color="#804040" size="5"><em>Gene</em></font><font size="4"> </b>(2000)
261, </font><font size="3">127-137</font><b></p>

<p ALIGN="CENTER"></b><font size="1">With copyright permission from Elsevier Science,
which allows single copies to be printed for personal use.</font><b></p>

<p ALIGN="CENTER"><font SIZE="2">&nbsp;</font><img src="images/colorbar.gif" alt="colorbar.gif (2362 bytes)" WIDTH="596" HEIGHT="6"></p>

<p ALIGN="left"><big><a href="#1. Certain structural principles">1. Certain structural
principles</a></big></p>

<p ALIGN="left"><big><a href="#2. The cluster rule">2. The cluster rule</a></big></p>

<p ALIGN="left"><big><a href="#3. The second parity rule">3. The second parity rule</a></big></p>

<p ALIGN="left"><big><a href="#4. Adaptive value of purine loading">4. Adaptive value of
purine loading</a></big></p>

<p ALIGN="left"><big><a href="#5. The GC rule">5. The GC rule</a></big></p>

<p ALIGN="left"><big><a href="#6. Codon choice">6. Codon choice</a></big></p>

<p ALIGN="left"><big><a href="#7. Thermophilic bacteria">7. Thermophilic bacteria</a></big></p>

<p ALIGN="left"><big><a href="#8. The &quot;holy grail&quot; of Romanes and Bateson">8.
The &quot;holy grail&quot; of Romanes and Bateson</a></big></p>

<p ALIGN="left"><a href="#9. The unpairing postulate"><big>9. The unpairing postulate</big></a></p>

<p ALIGN="left"><big><a href="#10. Universal Darwinism">10. Universal Darwinism?</a></big></p>

<p ALIGN="left"><img src="images/colorbar.gif" alt="colorbar.gif (2362 bytes)" WIDTH="596" HEIGHT="6"></p>

<p><font color="#800040" size="5">Abstract</font></p>
</b><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFF7F2">
  <tr>
    <td width="100%"><p align="justify"><font face="Arrus BT" size="5" color="#800080">O</font><font face="Arrus BT" size="4" color="#800080">f Chargaff's four rules on DNA base composition,
    only his </font><font face="Arrus BT" size="4" color="#FF0000">first parity rule</font><font face="Arrus BT" size="4" color="#800080"> was incorporated into mainstream biology as the
    DNA double helix. Now, the </font><font face="Arrus BT" size="4" color="#FF0000">cluster
    rule</font><font face="Arrus BT" size="4" color="#800080">, the </font><font face="Arrus BT" size="4" color="#FF0000">second parity rule</font><font face="Arrus BT" size="4" color="#800080">, and the </font><font face="Arrus BT" size="4" color="#FF0000">GC
    rule</font><font face="Arrus BT" size="4" color="#800080">, reveal the multiple levels of
    information in our genomes and potential conflicts between them. In these terms we can
    understand </font></p>
    <ul>
      <li><font face="Arrus BT" size="4" color="#800080">how double-stranded RNA became an
        intracellular alarm signal,<br>
        <br>
        </font></li>
      <li><font face="Arrus BT" size="4" color="#800080">how potentially recombining nucleic acids
        can distinguish between</font><font face="Arrus BT" size="4"> &quot;<font color="#0000A0">self</font>&quot;
        <font color="#800080">and</font> &quot;<font color="#0000A0">not-self</font>&quot; <font color="#800080">so leading to the origin of species, <br>
        <br>
        </font></font></li>
      <li><font color="#800080" face="Arrus BT" size="4">how isochores evolved to facilitate gene
        duplication, and<br>
        <br>
        </font></li>
      <li><font color="#800080" face="Arrus BT" size="4">how unlikely it is that any mutation can
        ever remain truly neutral.</font></li>
    </ul>
    <p align="justify"><i><font color="#008000"><strong>Keywords</strong></font>:</i> Base
    composition; Chargaff&#146;s rules; Double-stranded RNA; Self/Not-self discrimination;
    Speciation; Szybalski&#146;s transcription direction rule; Thermophiles</td>
  </tr>
</table>
</center></div>

<p><a name="1. Certain structural principles"><big><big><font color="#800040"><big>1.
Certain structural principles</big></font></big></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="6">F</font><font face="Bell MT" size="4">ifty years ago Erwin Chargaff and his colleagues noted &quot;<font color="#0000A0">regularities</font>&quot; in the base composition of nucleic acids, which
    they considered &quot;<font color="#0000A0">reflected the existence in all DNA
    preparations of certain structural principles</font>&quot; (Chargaff, 1950, 1951). In
    particular, for duplex DNA they identified a species-invariant component of the base
    composition, &#151; %A = %T and %C = %G. Shortly thereafter, this &quot;<font color="#FF0000">first parity rule</font>&quot; was dramatically confirmed by the
    Watson-Crick double-helix model (Watson and Crick, 1953). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;&nbsp; However, the
    Chargaff laboratory made three other fundamental observations on the base composition of
    DNA, which are only now being incorporated into mainstream biology. Two of these were
    species-invariant, but one varied with the species. The first species-invariant
    observation was that individual bases are clustered to a greater extent than expected on a
    random basis (the &quot;<font color="#FF0000">cluster rule</font>&quot;; Chargaff, 1963).
    Thus:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEA">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">Another consequence of our
    studies on deoxyribonucleic acids of animal and plant origin is the conclusion that at
    least 60% of the pyrimidines occur as oligonucleotide tracts [runs] containing three or
    more pyrimidines in a row; and a corresponding statement must, owing to the equality
    relationship [between the two strands], apply also to the purines</font>.&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify">&nbsp;&nbsp; <font face="Bell MT" size="4">&nbsp; The
    second species-invariant observation was that Chargaff's first parity rule also applies,
    to a close approximation, to single-stranded DNA (his &quot;<strong><font color="#FF0000">second
    parity rule</font></strong>&quot;). If the individual strands of a DNA duplex are isolated
    and their base compositions determined, then %<strong>A</strong> = %<strong>T</strong>,
    and %<strong>C</strong> = %<strong>G</strong> (Rudner et al., 1968). Thus it was noted
    that there is an:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEA">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">equality &#151; even in
    the separated DNA strands &#151; of 6-amino [<strong>A</strong> + <strong>C</strong>] and
    6-keto [<strong>G</strong> + <strong>T</strong>] nucleotides, in the absence of all other
    pairing regularities</font>&quot;.</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; The
    validity of the rule became clearer when full genome sequences became available. For
    example, the &quot;<font color="#0000A0">top</font>&quot; strand of Vaccinia virus has
    63921 <strong>A</strong>s, 63776 <strong>T</strong>s, 32010 <strong>C</strong>s, and 32030
    <strong>G</strong>s. The rule holds, albeit less precisely, even when sequences are
    divided into segments of a few hundred bases (Bell and Forsdyke, 1999a). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Finally, there was the
    observation that the ratio of <strong>C</strong> + <strong>G</strong> to the total bases (<strong>A+C+G+T</strong>)
    tends to be constant in a particular species, but varies between species (the &quot;<font color="#FF0000"><strong>GC rule</strong></font>&quot;; Chargaff, 1951, 1979).</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEA">
  <tr>
    <td width="100%">&quot;<font color="#0000A0">DNA is in its composition characteristic of
    the species from which it is derived. This can &#133; be demonstrated by determining the
    ratios in which the individual purines and pyrimidines occur &#133;. There appear to exist
    two main groups of&nbsp; DNA, namely the &#145;<strong>AT</strong> type,&#146; in which
    adenine and thymine predominate, and the &#145;<strong>GC</strong> type,&#146; in which
    guanine and cytosine are the major constituents</font>.&quot; </td>
  </tr>
</table>
</center></div><b>

<p></b>&nbsp;</p>

<p><a name="2. The cluster rule"><big><big><font color="#800040"><big>2. The cluster rule</big></font></big></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">T</font><font face="Bell MT" size="4">he cluster rule was extended by work from Waclaw Szybalski's
    laboratory showing that base clustering in microorganisms often relates to transcription
    direction. The &quot;<font color="#0000A0">top</font>&quot; strand of part of a DNA duplex
    which is transcribed contains pyrimidine clusters if transcription is to the left of the
    promoter, and purine clusters if transcription is to the right of the promoter (Szybalski
    et al., 1966). For the circular lambdaphage genome, since genes to the left of the origin
    of replication are transcribed to the left, and genes to the right of this origin are
    transcribed to the right, the distribution of clusters also relates to the origin of
    replication (Szybalski et al., 1969).</font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; The observation of
    base clustering did not necessarily imply a local conflict with Chargaff's second parity
    rule. For example, a run of <strong>T</strong> residues, might be accompanied by a
    corresponding number of dispersed <strong>A</strong> residues, so that %<strong>A</strong>
    = %<strong>T</strong>. However, Oliver Smithies and his colleagues (1981) showed that
    there are distinct local deviations from the second parity rule, which again correlate
    with transcription and replication directions. In the circular genome of SV40 virus it was
    noted that to the left of the origin of replication genes are transcribed to the left, and
    here %<strong>C</strong> &gt; % <strong>G</strong>, whereas to the right of the origin of
    replication genes are transcribed to the right, and here %<strong>G</strong> &gt; %<strong>C</strong>.
    </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Thus, clustering can
    result in local deviations from the second parity rule in favour of the clustered base.
    When transcription is to the left, the top strand is the mRNA template strand
    (pyrimidine-rich), and the bottom strand is the mRNA synonymous strand (purine-rich). When
    transcription is to the right, the top strand is the mRNA synonymous strand (purine-rich),
    and the bottom strand is the mRNA template strand (pyrimidine-rich). It follows that,
    whether arising from a gene transcribed to the left or to the right, mRNAs tend to be
    &quot;<font color="#0000A0">purine-loaded</font>&quot;. This may be expressed as the
    &quot;<font color="#0000A0">purine-loading index</font>&quot; (Lao and Forsdyke, 2000),
    which may be calculated from codon-usage tables (Nakamura et al., 1999; Fig. 1). Some
    intriguing exceptions (i.e. pyrimidine-loading) are discussed elsewhere (Cristillo et al.,
    1998; Bell and Forsdyke, 1999b; Forsdyke, 2001).</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="100%" bgcolor="#FFFFEA">
  <tr>
    <td width="100%" bgcolor="#FFFFFF" align="center"><img src="images/pli001.gif" alt="pli001.TIF (1132304 bytes)" WIDTH="545" HEIGHT="411"></td>
  </tr>
  <tr>
    <td width="100%"><p align="justify"><strong>Fig. 1</strong>. Distribution of
    purine-loading among species. Purine-loading of coding regions was calculated from codon
    usage tables for all species represented in the August 1999 release of the GenBank
    database by more than three genes and more than 2500 bases. The purine-loading index
    (bases/kb) for a particular species was calculated as the sum of 1000[(<strong>G</strong>-<strong>C</strong>)/<strong>N</strong>]
    and 1000[(<strong>A</strong>-<strong>T</strong>)/<strong>N</strong>], where <strong>G</strong>,
    <strong>C</strong>, <strong>A</strong>, and <strong>T</strong> correspond to the numbers
    of individual bases, and <strong>N</strong> corresponds to the total number of bases, in
    the codon usage table. This measure of the purine-loading of RNAs disregards 5&#146; and
    3&#146; non-coding sequences, including poly(A) tails. The value for all human genes
    (excluding mitochondria) is 42 bases/kb, meaning that, on average, there are 42 more
    purines than pyrimidines for every kilobase of coding sequence. The shoulder with negative
    purine-loading values (i.e. pyrimidine-loading) corresponds mainly to mitochondrial genes.</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;
    Smithies used non-overlapping &quot;<font color="#0000A0">windows</font>&quot; of approx.
    0.1 kb to examine base composition. Bases were counted in a window and then the window was
    moved along the sequence, and the count repeated. Thus a base-composition profile for a
    genomic region was constructed. When sequences of much larger genomes became available in
    the 1990s, Donald Forsdyke and colleagues showed that a window of 1 kb can identify more
    precisely the locations of genes (open reading frames) and their transcriptional
    orientation (Dang et al., 1998; Bell and Forsdyke, 1999a, b; Fig. 2). Furthermore, Jean
    Lobry showed that the relationship to the origin of replication is a feature of several
    microbial genomes (&quot;<font color="#0000A0">skew analysis</font>&quot;; Frank and
    Lobry, 1999; Rocha et al., 1999).</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="100%" bgcolor="#FFFFEC">
  <tr>
    <td width="100%" align="center" bgcolor="#FFFFFF"><img src="images/transdir.gif" alt="transdir.gif (7732 bytes)" WIDTH="415" HEIGHT="273"></td>
  </tr>
  <tr>
    <td width="100%"><p align="justify"><strong>Fig. 2</strong>. Szybalski's transcription
    direction rule, evaluated as &quot;<font color="#0000A0">Chargaff differences</font>&quot;
    (deviations from Chargaff's second parity rule). Heavy blue horizontal arrows refer to the
    &quot;<font color="#0000A0">top</font>&quot; and &quot;<font color="#0000A0">bottom</font>&quot;
    strands of duplex DNA. Grey boxes refer to intergenic DNA. Black balls represent RNA
    polymerases with thin horizontal arrows indicating the direction of transcription. In the
    case of leftward transcription the Chargaff difference for the top strand is in favour of
    pyrimidines (<strong>Y</strong>). In the case of rightward transcription the Chargaff
    difference for the top strand is in favour of purines (<strong>R</strong>). It follows
    that RNAs tend to be purine-loaded. When applied to uncharted DNA, Chargaff difference
    patterns provide clues to the location of genes and their transcriptional orientation.</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; Chargaff's
    main interest in the base cluster phenomenon was that, prior to the emergence of nucleic
    acid sequencing technology, it provided some measure of the uniqueness of the base order
    of a nucleic acid. Szybalski&#146;s main interest was the possibility that the clustering
    played a role in the control of transcription. This implied an evolutionary selection
    pressure for clustering so that organisms with clusters would better control transcription
    than organisms which did not have clusters. However, following a better understanding of
    Chargaff&#146;s second parity rule as a reflection of nucleic acid secondary structure
    (see Section 3), it was recognized that in many cases a selection pressure for clustering
    was likely to have arisen at the post-transcriptional level.</font></td>
  </tr>
</table>
</center></div>

<p>&nbsp;</p>
<b>

<p></b><big><big><a name="3. The second parity rule"><font color="#800040"><big>3. The
second parity rule</big></font></a> </big></big></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">C</font><font face="Bell MT" size="4">hargaff&#146;s first parity rule for duplex DNA was consistent
    with a base on one strand of the Watson-Crick duplex requiring a complementary base on the
    other strand of the duplex. By extrapolation, the existence of a parity rule for single
    strands of nucleic acid (Chargaff&#146;s second parity rule), suggested <font color="#0080C0"><strong><i>intrastrand</i></strong></font> base pairing. At least by
    virtue of the composition of the stems in stem-loop secondary structures there should be
    an approximate equivalence between the Chargaff base pairs. Do genomes have the potential
    to form such secondary structures? What adaptive forces (if any) could have created them?
    These questions began to be addressed when the genomic sequences of various bacterial
    viruses (bacteriophages) first became available in the 1970s. It became evident that
    genomes contain multiple levels of information, and that some forms of information
    conflict with others (Grantham, 1972; Grantham et al., 1985, 1986). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; In some phages the
    genome is RNA (e.g. R17, MS2), and in others the genome is DNA (e.g. T4). In classical
    Darwinian terms it was assumed that bacteriophages which encode optimal proteins are best
    adapted to their environment. Thus, the environment acting on virus proteins would have
    selected for survival the &quot;<font color="#0000A0">fittest</font>&quot; viruses whose
    genes encoded optimal proteins. However, it was observed that the base sequence seems to
    serve the needs of nucleic acid <font color="#0080C0"><strong><i>structure</i></strong></font>
    just as much as the protein-encoding function. Indeed, the needs of nucleic acid structure
    are sometimes served <font color="#0080C0"><strong><i>better</i></strong></font> than
    those of the protein-encoding function. Since bacteriophages do not encode rRNAs and
    tRNAs, the possibility arose that it was the structure of mRNAs which was of selective
    importance. Winston Salser noted in 1970:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFF0">
  <tr>
    <td width="100%"><p align="justify"><font color="#0000A0">&quot;RNA phage R17 has very
    extensive regions of highly ordered base pairing. It has seemed likely that this might be
    necessary to allow phage packaging. Bernice Ricard and I were therefore somewhat surprised
    to find that T4 messengers [mRNAs], which do not have to be packaged, also have a very
    large amount of secondary structure. &#133;Our results suggest that a high degree of
    secondary structure may be important in the functioning of most mRNA molecules. Because of
    the very high Tm's [temperature at which the &quot;melting&quot; of secondary structure is
    half maximum] we do not think that the base pairing seen is random. The possible functions
    of such extensive regions of base-pairing are unknown.&quot;</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">In 1972 Andrew Ball
    (Ball, 1972, 1973a,b) went further noting that:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEC">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">The selection pressure for
    specific base pairing in a messenger RNA severely limits its coding potential&quot;,
    &#133; so that &#133; &quot;there is a pressure for some amino acid sequences to be
    selected according to criteria which are distinct from the structure and function of the
    protein they constitute.</font>&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; This
    conclusion was supported by better algorithms for calculating RNA secondary structure
    (Jaeger et al., 1990), which showed that for many mRNA sequences the energetics of the
    folding of the natural sequence are better than those of the corresponding shuffled
    sequence (i.e. one would have to shuffle and fold many times to arrive by chance at a
    structure approaching the stability of the natural sequence; Le and Maisel, 1989;
    Forsdyke, 1995a; Seffens and Digby, 1999). It appeared that &quot;<font color="#0000A0">the
    hand of evolution</font>&quot; had arranged the order of bases to support mRNA structure,
    sometimes at the expense of the coding function. </font></p>
    <p align="justify"><font face=" Bell MT" size="4">&nbsp;&nbsp;&nbsp; Although abundantly
    present in the cytoplasm, tRNAs and rRNAs are encoded by a relatively small part of
    microbial genomes. In microorganisms the sequence of mRNAs, as an RNA entity, are more
    representative of the genome. If many mRNAs have highly significant secondary structure,
    then the corresponding genomic regions should also have this potential. Indeed, the
    primary evolutionary pressure for the elaboration of mRNA secondary structure might have
    been <font color="#0080C0"><strong><i>at the</i> <i>genomic level</i></strong></font>
    rather that at the mRNA level. If so, regions of a genome which are not transcribed into
    mRNAs might also demonstrate potential for secondary structure. </p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;&nbsp; When folding
    algorithms were applied to the sequences of individual DNA strands, it was found that
    there is indeed considerable potential for secondary structure. <font color="#FF0000">Stem-loop
    potential in DNA is not restricted to regions encoding mRNA (or rRNA or tRNA), but is also
    present in introns and in intergenic DNA.</font> Stem-loop potential, greater than that of
    the corresponding shuffled sequence, is diffusely distributed throughout the genomes of
    all species examined (Forsdyke, 1995a,b,c; Heximer et al., 1996). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; The conflict with the
    protein-encoding function was found to be particularly apparent in the case of genes
    evolving rapidly under positive Darwinian selection (Forsdyke, 1995b; 1996a). In these
    cases the pressure to adapt the protein sequence has been so powerful that base order has
    not been able to support stem-loop potential. The natural protein-encoding sequence has <font color="#0080C0"><strong><i>less</i> </strong></font>stem-loop potential than the
    corresponding shuffled sequence. Stem-loop potential is then diverted to introns, which
    are <font color="#0080C0"><strong><i>more conserved</i></strong></font> than the
    surrounding exons. Intron stem-loop potential is greater than in the corresponding
    shuffled sequence. This supports an explanation for the early origin of introns, which
    postulates that protein-encoding potential was imposed on prototypic genomes already
    supporting stem-loop potential (Forsdyke 1995a, b, c). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; At least by virtue of
    the stems in DNA stem-loop structures, it follows that there would have been an
    evolutionary pressure for single-strands of DNA to have approximate equivalences of the
    Watson-Crick pairing bases. Thus, Chargaff's second parity rule is consistent with single
    strands of DNA having considerable potential for forming secondary structure. The possible
    adaptive value of this secondary structure at the genomic level will be discussed after
    first considering the adaptive value of purine-loading at the post-transcriptional level.</font></font></td>
  </tr>
</table>
</center></div>

<p><a name="4. Adaptive value of purine loading"><big><big><font color="#800040"><big>4.
Adaptive value of purine loading</big></font></big></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">A</font><font face="Bell MT" size="4">n important implication of what is now called &quot;<font color="#0000A0">Szybalski's transcription direction rule</font>&quot; is that RNAs, in
    general, tend to be purine-loaded (Figs 1, 2). This observation is suggested by Chargaff's
    early work on the base composition of total RNA from various species, but his data would
    then have mainly reflected the compositions of the most abundant RNA form, the ribosomal
    RNAs (rRNAs; Chargaff, 1951; Elson &amp; Chargaff, 1955). When purine-loading was found to
    apply to RNAs in general, the possibility arose that the selection pressure for this might
    have arisen, not at the transcription level, but post-transcriptionally at the level of
    individual RNA molecules. Given that mRNAs tend<br>
    </font></p>
    <ul>
      <li><font face="Bell MT" size="4">(i) to be loaded with runs of purines (the cluster rule)
        and<br>
        <br>
        </font></li>
      <li><font face="Bell MT" size="4">(ii) to have an elaborate secondary structure (consistent
        with Chargaff&#146;s second parity rule), </font></li>
    </ul>
    <p align="justify"><font face="Bell MT" size="4">where in the structures would purine
    clusters be found? </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Since for base-pairing
    purine clusters must be matched with complementary pyrimidine clusters, and since
    pyrimidine clusters are scarce in mRNAs, purine clusters should occupy the unpaired
    regions of mRNA secondary structures, primarily the loops. Indeed, this is where they are
    found in calculated structures (Bell and Forsdyke, 1999b). Why should it be advantageous
    for an mRNA to &quot;<font color="#0000A0">load</font>&quot; its loops with purines?</font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; A possible answer to
    this question derived from the studies of Jun-ichi Tomizawa and his colleagues on the way
    &quot;<font color="#0000A0">sense</font>&quot; and &quot;<font color="#0000A0">antisense</font>&quot;
    RNA molecules interact, prior to forming a double-strand duplex molecule (dsRNA; Eguchi et
    al., 1991; Bull et al., 1998). It was found that sense sequences in RNA search out
    complementary antisense sequences through &quot;<font color="#0000A0">kissing</font>&quot;
    interactions between the tips of the loops of stem-loop structures. If Watson-Crick
    base-pairing between the loops is achieved, the interaction may then proceed to the
    formation of dsRNA. Thus, if RNAs were generally purine-loaded, &quot;<font color="#0000A0">kissing</font>&quot; interactions would decrease, and hence the
    probability of forming dsRNA would decrease. How could <font color="#0080C0"><strong><i>failure</i></strong></font>
    to form dsRNA be of selective advantage? This will be considered in two steps.<br>
    </font></p>
    <ul>
      <li><font face="Bell MT" size="4">First, we consider how the tendency to purine-load might
        have first arisen to prevent self-RNA&#150; self-RNA interactions.<br>
        <br>
        </font></li>
      <li><font face="Bell MT" size="4">Second, we consider how cells might have further exploited
        this tendency, in order to recognize non-self-RNAs. </font></li>
    </ul>
    <p align="justify"><font face="Bell MT" size="4">The &quot;<font color="#0000A0">distraction
    hypothesis</font>&quot; (Bell &amp; Forsdyke, 1999b) points out that the physico-chemical
    state of the &quot;<font color="#0000A0">crowded</font>&quot; cytosol (Fulton, 1982;
    Forsdyke, 1995d) is likely to be highly conducive to a most fundamental interaction, that
    between the codons of mRNAs and anticodons at the tips of loops in tRNAs. This results in
    transient formation of dsRNA segments, which normally would not exceed five base pairs
    (Bossi and Roth, 1980). However, a cytosolic environment which favours mRNA-tRNA kissing
    interactions would <i>also</i> favour mRNA-mRNA kissing interactions. This &quot;<font color="#0000A0">distraction</font>&quot; could have impeded mRNA translation and hence
    could have decreased the rate of protein synthesis. If, when not seriously conflicting
    with their coding role, mRNAs had accepted purine mutations in loop regions, then this
    &quot;<font color="#FF0000">purine-loading</font>&quot; would have diminished mRNA-mRNA
    interactions and increased the efficiency of protein synthesis. In some cases, so
    important has been the need to accept purine mutations, even the coding role appears to
    have been compromised (Rocha et al., 1999; Lao and Forsdyke, 2000). When this compromise
    is not possible, then the length of the protein can be increased by inserting simple
    sequence repeats of amino acids with purine-rich codons between functional domains
    (Cristillo et al., 1998; Forsdyke, 2001).</font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; The presence of
    mechanisms to <font color="#0080C0"><strong><i>avoid</i></strong></font> inadvertant
    formation of extensive segments of dsRNAs as a result of interactions between self-RNAs,
    leaves open the possibility of the evolution of mechanisms <font color="#0080C0"><strong><i>utilizing</i></strong></font>
    dsRNA as an intracellular alarm against not-self-RNAs. This was first recognized in the
    interferon response by which a cell infected by a virus communicates that fact to other
    cells of an organism. It was found that dsRNA was a most powerful inducer of interferon
    and of other antiviral adaptations (Isaacs, 1962; Ehrenfeld and Hunt, 1971; Marcus, 1983).
    However, Phillip Sharp (1999) noted that &quot;<font color="#0000A0">it remains a mystery
    how cells treated with interferon specifically suppress the translation of viral mRNAs in
    their cytoplasm and not cellular mRNAs</font>&quot;. Thus, <font color="#FF0000">as with
    so many problems in biology, the self/not-self discrimination aspect appears central</font>.
    Recently, dsRNA has been found to mediate intracellular alarms in a variety of plants and
    animals infected with intracellular pathogens; these pathogens, in turn, have evolved
    mechanisms to inactivate components of the alarm system (Voinnet et al., 1999; Fire,
    1999). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Although the mechanism
    of dsRNA formation is still uncertain (Hamilton and Baulcombe, 1999), one explanation for
    how an exogenous agent is recognised as not-self and triggers the formation of dsRNA
    challenges our usual way of regarding the mRNA population of a cell. This population seems
    obviously heterogeneous because it consists of multiple mRNA species each encoding a
    different product, &#151; a powerful viewpoint which preempts the search for alternative
    explanations. </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;&nbsp; However, if we
    consider the possibility that heterogeneity can serve more than one purpose, then we can
    regard the heterogenous <font color="#0080C0"><strong><i>intracellular</i></strong></font>
    mRNA population in the same way as we regard the heterogenous <font color="#0080C0"><strong><i>extracellular</i></strong></font>
    immunoglobulin population. The latter constitutes the first barrier against foreign
    pathogens (&quot;<font color="#FF0000">not-self</font>&quot;), which select from among the
    immunoglobulins (which have been randomly generated and then pre-selected for non-reaction
    with self antigens) those with specificity for the coat antigens of the pathogen. </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; In the case of a virus, the
    coat is relinquished at the time of entry into a cell, and the cell&#146;s first possible
    line of intracellular defence would be to recognize the foreign nucleic acid as &quot;<font color="#0000A0">not-self</font>&quot;, either in its genomic form, or in the form of an
    early transcript. This implicates dsRNA. Perhaps a segment of a particular host mRNA
    species, because it happened to have sufficient sequence complementarity, would form a
    double-stranded segment with viral RNA of sufficient length (at least two helical turns)
    to trigger an alarm response (Izant and Weintraub, 1984; Melton, 1985; Robertson and
    Mathews, 1996; Tian et al., 2000). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Thus, the heterogenous
    mRNA population can be seen as consisting of &quot;<font color="#0000A0">RNA antibodies</font>&quot;,
    which have been preselected over evolutionary time not to react with &quot;<font color="#0000A0">self</font>&quot; RNAs, while retaining, and if possible developing, the
    potential to react with &quot;<font color="#0000A0">not-self</font>&quot; RNAs. Since, due
    to failure of transcription termination, a low level of transcription of extra-genic DNA
    is possible (Heximer et al., 1998), the maximum potential repertoire of &quot;<font color="#0000A0">RNA antibodies</font>&quot; is limited only by genome size. A second line
    of intracellular defence would be recognition of the translation products of virus genes;
    a mechanism for this form of self/not-self discrimination is discussed elsewhere
    (Forsdyke, 1995d).</font></td>
  </tr>
</table>
</center></div>

<p>&nbsp;</p>

<p><big><big><a name="5. The GC rule"><font color="#800040"><big>5. The GC rule</big></font></a>
</big></big></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">W</font><font face="Bell MT" size="4">e propose above that in some circumstances evolutionary selective
    pressures have acted to preserve nucleic acid secondary structure, sometimes at the
    expense of an encoded protein. That this might also apply to the species-dependent
    component of the base composition, (<strong>C</strong>+<strong>G</strong>)%, arose from
    Naboru Sueoka's demonstration in 1961, before the genetic code was deciphered, that the
    amino acid composition of the proteins of microorganisms is influenced, not just by the
    demands of the environment on the proteins, but also by the base composition of the genome
    encoding those proteins. The observation has since been abundantly confirmed in a wide
    variety of animal and plant species (Lobry, 1997). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; Sueoka (1961) further
    pointed out that for individual &quot;<font color="#0000A0">strains</font>&quot; of <i>Tetrahymena</i>
    the (<strong>C+G</strong>)% (referred to as &quot;<strong>GC</strong>&quot;) tends to be
    uniform throughout the genome:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEA">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">If one compares the
    distribution of DNA molecules of <i>Tetrahymena</i> strains of different mean <strong>GC</strong>
    contents, it is clear that the difference in mean values is due to a rather uniform
    difference of <strong>GC</strong> content in individual molecules. In other words,
    assuming that strains of <i>Tetrahymena </i>have a common phylogenetic origin, when the <strong>GC</strong>
    content of DNA of a particular strain changes, all the molecules undergo increases or
    decreases of <strong>GC</strong> pairs in similar amounts. This result is consistent with
    the idea that the base composition is rather uniform not only among DNA molecules of an
    organism, but also with respect to different parts of a given molecule</font>.&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify">&nbsp;&nbsp; <font face="Bell MT" size="4">Again, this
    observation has been abundantly confirmed for a wide variety of species (Muto and Osawa,
    1987), although many organisms considered higher on the evolutionary scale have their
    genomes sectored into regions of low or high (<strong>C+G</strong>)% (Bernardi and
    Bernardi, 1986; Bernardi 2000; see Section 9).</font></p>
    <p><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Sueoka (1961) also noted a link
    between (<strong>C+G</strong>)% and reproductive isolation for strains of <i>Tetrahymena</i>:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFE8">
  <tr>
    <td width="100%"><p align="justify"><font color="#FF0000">&quot;DNA base composition is a
    reflection of phylogenetic relationship. Furthermore, it is evident that those strains
    which mate with one another (i.e. strains within the same 'variety') have similar base
    compositions. Thus strains of variety 1 ..., which are freely intercrossed, have similar
    mean <strong>GC</strong> content.&quot;</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; When the
    genetic code was deciphered in the early 1960s, it was observed that there are more codons
    than amino acids, so that most amino acids can correspond to more than one triplet codon.
    This gives some flexibility to a nucleic acid sequence. Sometimes an amino acid can be
    encoded from among as many as six possible synonymous codons. Walter Fitch (1974) noted
    that &quot;<font color="#0000A0">the degeneracy of the genetic code provides an enormous
    plasticity to achieve secondary structure without sacrificing specificity of the message</font>&quot;.
    </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Yet, as outlined
    above, sometimes even this &quot;<font color="#0000A0">plasticity</font>&quot; is
    insufficient, so that, with the exception of genes under positive Darwinian selection
    (Forsdyke, 1995b, 1996a), genomic secondary structure (&quot;<font color="#0000A0">fold
    pressure</font>&quot;) and (C+G)% &quot;<font color="#0000A0">call the tune</font>&quot;.
    Non-synonymous codon changes modify the amino acid sequence, sometimes at the <font color="#0080C0"><strong><i>expense</i></strong></font> of protein structure and function.
    A protein has to adapt to the demands of the environment, but it also has to adapt to
    genomic forces which we will show have derived, not from the conventional environment
    acting upon the convention (&quot;<font color="#0000A0">classical</font>&quot;) phenotype,
    but from what we call the &quot;<font color="#0000A0">reproductive environment</font>&quot;
    acting on the &quot;<font color="#0000A0">genome phenotype</font>'', or &quot;<font color="#0000A0">reprotype</font>&quot;. Thus Bernardi and Bernardi noted in 1986 that:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFF2">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&quot;<font color="#0000A0">The organismal phenotype comprises two components, the </font><font color="#FF0000">classical phenotype</font><font color="#0000A0">, corresponding to the
    'gene products', and a '</font><font color="#FF0000">genome phenotype</font><font color="#0000A0">' which is defined by [base] compositional constraints.</font></font>&quot;</td>
  </tr>
</table>
</center></div>

<p><b>&nbsp;</p>

<p></b><a name="6. Codon choice"><big><big><font color="#800040"><big>6. Codon choice</big></font></big></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">T</font><font face="Bell MT" size="4">he issue of which codon was employed in a particular circumstance
    was considered by Richard Grantham, who noted in 1972 that codon choice was not random in
    microorganisms, &quot;<font color="#0000A0">suggesting a mechanism against [base]
    composition drift</font>&quot;. Observing that &quot;<font color="#0000A0">little latitude
    appears left for &#145;neutral&#146; or synonymous mutations in coliphage codons</font>&quot;,
    he was led to his &quot;<font color="#FF0000">genome hypothesis</font>&quot;, which
    specified that undefined adaptive genomic pressure(s) caused changes in base composition
    and hence in codon choice (Grantham et al., 1986):</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEA">
  <tr>
    <td width="100%"><align="justify">&quot;<font color="#0000A0">Each ... species has a 'system' or coding
    strategy for choosing among synonymous codons. This system or dialect is repeated in each
    gene of a genome and hence is a characteristic of the genome.</font>&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; There was
    also a sense that the coding strategy was of relevance to the most fundamental aspects of
    an organism's biology:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFD9">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">What is the fundamental
    explanation for interspecific variation in coding strategy? Are we faced with a situation
    of continuous variation within and between species, thus embracing a Darwinian perspective
    of gradual separation of populations to form new species &#133;? This is the heart of the
    problem of molecular evolution</font>.&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;
    Grantham and his colleagues further pointed to the need to determine &quot;<font color="#FF0000">how much independence exists between the two levels of evolution</font>&quot;
    (that of the genome phenotype and of the classical phenotypic) and considered &quot;<font color="#FF0000">it is too easy just to say most mutations are neutral</font>&quot;.
    However, non-adaptive &quot;<font color="#0000A0">neutralist</font>&quot; explanations
    gained much support (Filipski, 1990; Sueoka, 1995). Paul Sharp and his colleagues
    concluded in 1993 that the main factors influencing codon choice are mutational biases and
    the need for highly expressed genes to be efficiently translated. Although phenotypic
    adaptive factors such as the need to translate an abundant mRNA efficiently can influence
    codon choice, genomic factors, identified here as stem-loop potential (&quot;<font color="#0000A0">fold pressure</font>&quot;) and (<strong>C+G</strong>)%, play an important
    and often dominant role.</font></td>
  </tr>
</table>
</center></div>

<p>&nbsp;</p>
<b>

<p></b><a name="7. Thermophilic bacteria"><big><font color="#800040"><big><big>7.
Thermophilic bacteria</big></big></font></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">T</font><font face="Bell MT" size="4">he secondary structure of nucleic acids with a high (<strong>C+G</strong>)%
    is more stable than that of nucleic acids with a low (<strong>C+G</strong>)%. <strong>GC</strong>
    bonds are associated with a more stable nucleic acid structure than <strong>AT</strong> or
    <strong>AU</strong> bonds. This is reflected in the base composition of RNAs whose
    structure is vital for their function, namely rRNAs and tRNAs. Free of coding constraints,
    yet required to form part of the precise structure of ribosomes, rRNAs might more readily
    accept mutations which increase <strong>GC</strong> content than do mRNAs. Indeed, the <strong>GC</strong>
    content of rRNAs is directly proportional to the normal growth temperature, so that rRNAs
    of thermophilic bacteria are highly enriched in <strong>G</strong> and <strong>C</strong>
    (Dalgaard and Garret, 1993; Forterre and Elie, 1993; Galtier and Lobry, 1997). However,
    although optimum growth temperature correlates positively with the <strong>GC</strong>
    content of rRNA, it does not correlate similarly with the <strong>GC</strong> content of
    genomic DNA, and hence with that of the mRNA populations transcribed from that DNA. </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; The finding of no
    consistent trend towards a high genomic <strong>GC</strong> in thermophilic organisms has
    been interpreted as supporting the neutralist argument that variations in genomic <strong>GC</strong>
    are the consequences of mutational biases and are, in themselves, of no adaptive value
    (Filipski 1990; Galtier and Lobry 1997). However, the finding is also consistent with the
    argument that genomic <strong>GC</strong> is too important merely to follow the dictates
    of temperature, since its primary role is related to other more fundamental adaptations
    (Bernardi and Bernardi, 1986). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Galtier and Lobry
    (1997) have argued that &quot;<font color="#0000A0">any secondary structure that must
    endure high temperatures requires a high <strong>G</strong> + <strong>C</strong> content</font>&quot;.
    This would include both the classical Watson-Crick secondary structure involving
    inter-strand base pairing, and any secondary structure involving intra-strand base pairing
    (Murchie et al., 1992). However, the stability of genomic DNA at high temperatures might
    be achieved in ways other than by an increase in <strong>GC</strong> content (Bernardi,
    2000). These include association with polyamines (Oshima et al., 1990), and relaxation of
    supercoiling (Friedman et al., 1995). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;&nbsp; There is no
    reason to believe that in thermophiles DNA is not able to maintain both its classical
    duplex structure with H-bonding between opposite strands, and any secondary structures
    involving intrastrand H-bonding. As we propose (see Section 9), the latter structures
    would be critical only under certain clearly defined, but selectively very important,
    circumstances, namely when recombination repair is required. The most enduring DNA
    secondary structure, even at high temperatures, would be the classical duplex form.</font></td>
  </tr>
</table>
</center></div>

<p>&nbsp;</p>

<p><a name="8. The &quot;holy grail&quot; of Romanes and Bateson"><big><font color="#800040"><big><big>8. The &quot;holy grail&quot; of Romanes and Bateson</big></big></font></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">W</font><font face="Bell MT" size="4">ith hindsight it seems that, in identifying (<strong>C+G</strong>)%
    as the species variant component of the base composition, Chargaff had uncovered what we
    might now recognize as the &quot;<font color="#0000A0">holy grail</font>&quot; of
    speciation first postulated in 1886 by Charles Darwin's research associate, George Romanes
    (Forsdyke, 1999a, b). Romanes had pointed to what we would now call non-genic variations
    in the germ-line, which would tend to isolate an individual reproductively from other
    members of its species, but not from members that had undergone the same variation.
    William Bateson further postulated a non-genic inherited variation, which would remain
    constant for a species, whereas genic variations could occur within a species. The
    non-genic variations, in whatever was responsible for carrying hereditary information from
    generation to generation (not known at that time), would have the potential to lead to
    species differentiation, so that variant members of a species (&quot;<font color="#0000A0">not-self</font>&quot;)
    would not successfully reproduce with members of the main species (&quot;<font color="#0000A0">self</font>&quot;). The latter would constitute the &quot;reproductive
    environment&quot; <i><font color="#0080C0"><strong>moulding</strong></font> </i>the genome
    phenotype (reprotype). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Once reproductive
    isolation was achieved, the natural selection postulated by Darwin would be able to
    further increase species differentiation by allowing the survival of organisms with
    advantageous genic variations, and disallowing the survival of organisms with
    disadvantageous genic variations. These genic variations would affect the classical
    phenotype. Romanes referred to his holy grail (speciating factor) as an &quot;<font color="#0000A0">intrinsic peculiarity</font>&quot; of the reproductive system. Bateson
    described his holy grail as a speciating factor uniformly attached to the same &quot;<font color="#0000A0">residue</font>&quot; as the genes, but distinct from the genes. These are
    just the properties we find in the (<strong>C+G</strong>)% (Forsdyke, 1996b, 1998).</font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; A metaphor for the
    role (<strong>C+G</strong>)% might play in keeping individuals reproductively isolated
    from each other is provided by the word &quot;<font color="#0000A0">dialect</font>&quot;
    (Grantham et al., 1986). A common language brings people together, and in this way is
    conducive to sexual reproduction. But languages can vary, first into dialects and then
    into independent sub-languages. Linguistic differences keep people apart, and this
    difference in the reproductive environment militates against sexual reproduction.</font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; At the molecular
    level, we see similar forces acting at the level of meiosis. Here paternal and maternal
    chromosomal homologs align. Forsdyke has proposed that if there is sufficient sequence
    identity, so that the DNA &quot;<font color="#0000A0">dialects</font>&quot; [(<strong>C+G</strong>)%]
    match, meiosis is likely to progress through various check-points (Page and Orr-Weaver,
    1986), and gametes will be formed. If there is insufficient identity (i.e. the DNA &quot;<font color="#0000A0">dialects</font>&quot; do not match) meiosis will fail, gametes will not
    form, and the individual will be sterile, &#151; a &quot;<font color="#0000A0">mule</font>&quot;.
    Thus, the original parental lines will be reproductively isolated from each other, and as
    such would be defined as distinct species. Changes in the (<strong>C+G</strong>)% dialects
    have the potential to initiate speciation (Forsdyke, 1996b).</font></td>
  </tr>
</table>
</center></div>

<p><b>&nbsp;</p>

<p></b><a name="9. The unpairing postulate"><big><font color="#800040"><big><big>9. The
unpairing postulate</big></big></font></big></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">I</font><font face="Bell MT" size="4">n 1922 Muller suggested that the pairing of genes as parts of
    chromosomes undergoing meiotic synapsis, might provide clues to gene structure and
    replication:</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEC">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">It is evident that the
    very same forces which cause the genes to grow should also cause like genes to attract
    each other, ... If the two phenomena are thus dependent on a common principle in the
    make-up of the gene, progress made in the study of one of them should help in the solution
    of the other.</font>&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; In
    1954 he set his students an essay &quot;<font color="#0000A0">How does the Watson-Crick
    model account for synapsis?</font>&quot; (Carlson, 1981). Crick took up the challenge in
    1971 with his &quot;<font color="#FF0000">unpairing postulate</font>&quot; by which the
    two strands of the classical DNA duplex would unpair to allow a homology search.</font></p>
    <p align="justify">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <font face="Bell MT" size="4">Recent
    work on the meiotic alignment of chromosomes suggests that initiation of speciation
    involves that aspect of the genome phenotype (reprotype), which is constituted by the
    genome-wide potential to extrude stem-loops. Those of the maternal and paternal
    chromosomal homologs may mutually explore each other and test for &quot;<font color="#0000A0">self</font>&quot; DNA complementarity, using the &quot;<font color="#0000A0">kissing</font>&quot; mechanism of Tomizawa (Eguchi et al., 1991; Hawley
    and Arbel, 1993; Kleckner, 1997). If sufficient complementarity is found (i.e. the genomes
    are reprotypically compatible), then crossing over and recombination can occur. The main
    adaptive value of this would be to provide for the correction of errors in the individual
    homologs (Winge, 1917; Bernstein and Bernstein, 1991). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Where does the (<strong>C+G</strong>)%
    &quot;<font color="#0000A0">dialect</font>&quot; come into this? It has been observed that
    small fluctuations in (<strong>C+G</strong>)% would have a major effect on the ability of
    duplex DNA molecules to extrude stem-loops and on the pattern of loops which then occur. A
    very small difference in (<strong>C+G</strong>)% (reprotypic difference) would mark a
    meiotically pairing DNA as &quot;<font color="#0000A0">not-self</font>&quot;. This would
    impair the kissing interaction with &quot;<font color="#0000A0">self</font>&quot; DNA
    (Forsdyke 1998; Bull et al., 1998), and so would disrupt meiosis and allow divergence
    between the two parental lines, thus initiating speciation (Forsdyke, 1996b). Consistent
    with this, direct tests of incipient speciation in the fruit fly (the phenomenon known as
    Haldane&#146;s rule) implicate differences in DNA <i>per se</i>, rather than in distinct
    genes, as initiating reproductive isolation (Naveira and Maside, 1998; Forsdyke, 2000a).
    Similar considerations may apply to the phenomenon of &quot;<font color="#0000A0">heteroduplex
    resistance</font>&quot; between recombining DNAs of different bacterial species (Majewski
    and Cohan, 1998). </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; Once a speciation
    process has begun, prezygotic factors and postzygotic factors other than (<strong>C+G</strong>)%
    are likely to replace the original difference in (<strong>C+G</strong>)% as a barrier to
    reproduction (i.e. a barrier to recombination). In this circumstance, (<strong>C+G</strong>)%
    becomes free to adopt other roles, such as the prevention of intragenomic recombination.
    This could involve intragenomic differentiation of regions of high and low (<strong>C+G</strong>)%
    (isochores). These have the potential to &quot;<font color="#0000A0">reproductively
    isolate</font>&quot; (i.e. recombinationally isolate) different parts of the genome. Thus,
    the attempted duplication of a globin gene into alpha-globin and beta-globin genes might
    have failed since sequence similarity would favour recombination between the two genes and
    any incipient differences would be eliminated. However, the duplication appears to have
    involved relocation to a different isochore with corresponding changes in (<strong>C+G</strong>)%,
    so that the two genes became recombinationally isolated. </font></p>
    <p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp;&nbsp; As a consequence
    of the differences in (<strong>C+G</strong>)% the corresponding mRNAs utilize different
    codons for corresponding amino acids, even though both mRNAs are translated in the <font color="#0080C0"><strong><i>same</i></strong></font> cell using the <font color="#0080C0"><strong><i>same</i></strong></font>
    ribosomes and tRNA populations. Thus, it is unlikely that the primary pressure to
    differentiate codons arose at the translational level (Grantham, 1986). Isochores would
    have arisen as a random fluctuation in base composition in a genomic region such that one
    product of a gene duplication was able to survive for a sufficient number of generations
    to allow functional differentiation to occur. The regional base compositional fluctuation
    would then have &quot;<font color="#0000A0">hitch-hiked</font>&quot; through the
    generations on the successful duplicate.</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="100%" bgcolor="#FFFFF2">
  <tr>
    <td width="100%" align="center"><img src="images/genopres.gif" alt="genopres.TIF (1027710 bytes)" WIDTH="536" HEIGHT="396"></td>
  </tr>
  <tr>
    <td width="100%"><strong>Fig. 3</strong>. Summary of potentially conflicting evolutionary
    pressures as manifest at the level of mRNA (red line with arrow-head). <ul>
      <li>1. <font color="#FF0000">(<strong>C+G</strong>)% pressure</font> (&quot;<font color="#0000A0"><strong>GC</strong> pressure</font>&quot;) acting primarily at the genomic
        level, and secondarily affecting mRNA base<i> <font color="#0080C0"><strong>composition</strong></font></i>.<br>
        <br>
      </li>
      <li>2. <font color="#FF0000">Fold (stem-loop) pressure</font> acting primarily at the
        genomic level and secondarily affecting mRNA base<i> <font color="#0080C0"><strong>order</strong></font></i>.<br>
        <br>
      </li>
      <li>3. <font color="#FF0000">Purine pressure</font> acting primarily at the cytoplasmic
        level to enrich loops with purines. <br>
        <br>
      </li>
      <li>4. <font color="#FF0000">Coding pressure</font> deriving from classical environmental
        interactions with the conventional phenotype, which result in base changes in the
        protein-encoding part of the mRNA. <br>
        <br>
      </li>
      <li>5. <font color="#FF0000">Regulatory pressures</font> (small violet boxes) acting
        primarily at the cytoplasmic level, which result in base changes mainly in the 5' and 3'
        non-coding regions.</li>
    </ul>
    </td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp;&nbsp; The
    multiple evolutionary pressures acting on mRNAs and genomes are summarized in Figure 3 and
    Table 1. It should be noted that changes in genetic fitness (changes in the number of
    progeny transmitted to future generations), could result from changes in the classical
    phenotype and/or in the genome phenotype. A mutation which appeared neutral with respect
    to protein function might nevertheless affect fitness by changing the genome phenotype. It
    is argued elsewhere that the apparently neutral polymorphism of some intracellular
    proteins is an adaptation to facilitate intracellular self/not-self discrimination
    (Forsdyke, 2000b).</font></td>
  </tr>
</table>
</center></div><div align="center"><center>

<table BORDER="1" CELLSPACING="1" CELLPADDING="7" WIDTH="100%" bgcolor="#FFE8DD" bordercolor="#FF8040">
  <tr>
    <td VALIGN="TOP" COLSPAN="5" width="549" height="16"><font size="2"><font color="#800040"><em><strong>Table
    1</strong></em></font>&nbsp; <font color="#004080"><b>Postulated evolutionary processes
    leading to multiple levels of information in genomes</b></font></font></td>
  </tr>
  <tr>
    <td WIDTH="109" VALIGN="TOP" height="60"><b><font size="2">Environmental selective factors</font></b></td>
    <td WIDTH="86" VALIGN="TOP" height="60"><b><font size="2">Selection for mutations which:</font></b></td>
    <td WIDTH="78" VALIGN="TOP" height="60"><b><font size="2">Primary effect on DNA function</font></b></td>
    <td WIDTH="100" VALIGN="TOP" height="60"><b><font size="2">Biological result</font></b></td>
    <td WIDTH="123" VALIGN="TOP" height="60"><b><font size="2">Observed features of modern DNA</font></b></td>
  </tr>
  <tr>
    <td VALIGN="TOP" COLSPAN="5" width="549" height="1"></td>
  </tr>
  <tr>
    <td WIDTH="109" VALIGN="TOP" height="75"><font size="2">Classical phenotypic selective
    factors</font></td>
    <td WIDTH="86" VALIGN="TOP" height="75"><font size="2">Change encoded proteins</font></td>
    <td WIDTH="78" VALIGN="TOP" height="75"><font size="2">None. </font><p>&nbsp;</td>
    <td WIDTH="100" VALIGN="TOP" height="75"><font size="2">Change in classical phenotype</font></td>
    <td WIDTH="123" VALIGN="TOP" height="75"><font size="2">Usually change in a base pair in
    accordance with Chargaff&#146;s first parity rule</font></td>
  </tr>
  <tr>
    <td WIDTH="109" VALIGN="TOP" height="79"><font size="2">Competitors with more efficient
    translation. </font></td>
    <td WIDTH="86" VALIGN="TOP" height="79"><font size="2">Purine-load RNAs </font></td>
    <td WIDTH="78" VALIGN="TOP" height="79"><font size="2">None. </font><p>&nbsp;</td>
    <td WIDTH="100" VALIGN="TOP" height="79"><font size="2">Efficient translation with no
    &quot;self&quot; dsRNA formation</font></td>
    <td WIDTH="123" VALIGN="TOP" height="79"><font size="2">Chargaff's cluster rule and
    Szybalski's transcription direction rule</font></td>
  </tr>
  <tr>
    <td WIDTH="109" VALIGN="TOP" height="75"><font size="2">Mutagens</font></td>
    <td WIDTH="86" VALIGN="TOP" height="75"><font size="2">Promote DNA stem-loop potential</font></td>
    <td WIDTH="78" VALIGN="TOP" height="75"><font size="2">Within species meiotic
    recombination promoted</font></td>
    <td WIDTH="100" VALIGN="TOP" height="75"><font size="2">Change in genome phenotype for DNA
    repair</font></td>
    <td WIDTH="123" VALIGN="TOP" height="75"><font size="2">Chargaff's second parity rule</font></td>
  </tr>
  <tr>
    <td WIDTH="109" VALIGN="TOP" height="132"><font size="2">Recombinationally
    &quot;not-self&quot; sexual partners</font></td>
    <td WIDTH="86" VALIGN="TOP" height="132"><font size="2">Impair homology search between
    DNAs of species members whose sequences are diverging</font></td>
    <td WIDTH="78" VALIGN="TOP" height="132"><font size="2">Meiotic recombination is impaired </font></td>
    <td WIDTH="100" VALIGN="TOP" height="132"><font size="2">Change in genome phenotype for
    speciation</font></td>
    <td WIDTH="123" VALIGN="TOP" height="132"><font size="2">Chargaff's GC rule</font></td>
  </tr>
</table>
</center></div><font SIZE="4">

<p>&nbsp;</p>
</font>

<p><a name="10. Universal Darwinism"><font color="#800040" SIZE="4"><big><big>10.
Universal Darwinism?</big></big></font></a></p>
<div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="5">C</font><font face="Bell MT" size="4">hargaff was well aware of the evolutionary implications of his
    work, writing of &quot;<font color="#0000A0">the survival of the fittest nucleic acids</font>&quot;
    (Chargaff, 1951). However, Biology satisfied with what was called &quot;<font color="#0000A0"><i>the</i> modern synthesis</font>&quot; (Carlson, 1981), had not kept
    pace with Chemistry, and it was difficult for him to explore the implications of his
    discovery (Chargaff, 1979). Romanes and Bateson, who had challenged Darwinian dogma, had
    been dismissed by generations of biologists bewitched by genes and all that they promised.
    To put it mildly, the level of discourse was not constructive. </font></p>
    <ul>
      <li><font face="Bell MT" size="4">Systematist Ernst Mayr (1980) when expressing &quot;<font color="#0000A0">some thoughts on the history of the evolutionary synthesis</font>&quot;
        classified Bateson as among those geneticists who failed to understand evolution (implying
        that others understood it better). <br>
        <br>
        </font></li>
      <li><font face="Bell MT" size="4">The plant geneticist Ledyard Stebbins (1980) commenting on
        &quot;<font color="#0000A0">botany and the synthetic theory of evolution</font>&quot;
        blamed Bateson for &quot;<font color="#0000A0">delaying the synthesis</font>&quot;. <br>
        <br>
        </font></li>
      <li><font face="Bell MT" size="4">The &quot;<font color="#0000A0">universal Darwinist</font>&quot;
        Richard Dawkins lamenting the position he perceived to have been taken by the &quot;<font color="#0000A0">mutationists of the early part of this century</font>&quot; noted (1983)
        that:</font></li>
    </ul>
    </td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="80%" bgcolor="#FFFFEC">
  <tr>
    <td width="100%"><p align="justify">&quot;<font color="#0000A0">For historians there
    remains the baffling enigma of how such distinguished biologists as &#133;W. Bateson
    &#133; could rest satisfied with such a crassly inadequate theory. ... The irony with
    which we must now read W. Bateson's dismissal of Darwin is almost painful</font>.&quot;</td>
  </tr>
</table>
</center></div><div align="center"><center>

<table border="0" cellpadding="25" cellspacing="0" width="90%">
  <tr>
    <td width="100%"><p align="justify"><font face="Bell MT" size="4">&nbsp;&nbsp; Despite the
    overwhelming rhetoric to the contrary, we are now beginning to appreciate the deep truths
    which Romanes and Bateson were attempting to convey (Forsdyke, 1999a,b; 2000a). When
    Bateson died in 1926, Chargaff was twenty. He is still with us, &#150; a citizen of the
    twenty first century. We wish him well. </font></td>
  </tr>
</table>
</center></div><b>

<p><font color="#800080">Acknowledgements</font></b></p>

<p>&nbsp;&nbsp;&nbsp; We thank Richard Grantham and Tim Greenland for a very helpful
review of the manuscript, and Jim Gerlach for assistance with computer configuration.
Academic Press, Cold Spring Harbor Laboratory Press, Elsevier Science and the National
Research Council of Canada gave permission to place full-text versions of some of the
above cited papers in Forsdyke's web pages.</p>
<b>

<p></b>&nbsp;</p>

<p><font color="#800040"><b>References</b></font></p>

<p>Ball, L. A. 1972. Implications of secondary structure in messenger RNA. <font color="#804040"><em><strong>J. Theor. Biol</strong></em></font>. <strong>36</strong>,
313-320.</p>

<p>Ball, L. A. 1973a. Secondary structure and coding potential of the coat protein gene of
bacteriophage MS2. <font color="#804040"><em><strong>Nature New Biol</strong></em></font>.
<strong>242</strong>, 44-45. </p>

<p>Ball. L. A. 1973b. Mutual influence of the secondary structure and information content
of a messenger RNA. <font color="#804040"><em><strong>J. Theor. Biol</strong></em></font>.
<strong>41</strong>, 243-247.</p>

<p>Bell, S. J., Forsdyke, D. R. 1999a. Accounting units in DNA. <font color="#804040"><em><strong>J.
Theor. Biol</strong></em></font>. <strong>197</strong>, 51-61.</p>

<p>Bell, S. J., Forsdyke, D. R. 1999b. Deviations from Chargaff's second parity rule
correlate with direction of transcription. <font color="#804040"><em><strong>J. Theor.
Biol</strong></em></font>. <strong>197</strong>, 63-76.</p>

<p>Bernardi G., Bernardi, G. 1986. Compositional constraints and genome evolution. <font color="#804040"><em><strong>J. Mol. Evol</strong></em></font>. <strong>24</strong>, 1-11.</p>

<p>Bernardi G. 2000. Isochores and the evolutionary genomics of vertebrates. <font color="#804040"><em><strong>Gene</strong></em></font> <strong>241</strong>, 3-17.</p>

<p>Bernstein, C., Bernstein, H. 1991. <font color="#804040"><em><strong>Aging, Sex and DNA
Repair</strong></em></font>. Academic Press, San Diego.</p>

<p>Bossi, L., Roth, J. R. 1980. The influence of codon context on genetic code
translation. <font color="#804040"><em><strong>Nature</strong></em></font><i> </i><strong>286</strong>,
123-127.</p>

<p>Bull, J. J., Jacobson, A., Badgett, M. R., Molineux, I. J. 1998. Viral escape from
antisense RNA. <font color="#804040"><em><strong>Molec. Microbiol</strong></em></font>. <strong>28</strong>,
835-846.</p>

<p>Carlson, E. A. 1981. <font color="#804040"><em><strong>Genes, Radiation and Society.
The Life and Work of H. J. Muller. </strong></em></font>Cornell University Press, Ithaca,
NY, pp. 390-392.</p>

<p>Chargaff, E. 1950. Chemical specificity of nucleic acids and mechanism of their enzymic
degradation. <font color="#804040"><em><strong>Experientia</strong></em></font> <strong>6</strong>,
201-209.</p>

<p>Chargaff, E. 1951. Structure and function of nucleic acids as cell constituents. <font color="#804040"><em><strong>Fed. Proc</strong></em></font>. <strong>10</strong>, 654-659.</p>

<p>Chargaff, E. 1963. <font color="#804040"><em><strong>Essays on Nucleic Acids</strong></em></font>.
Elsevier, Amsterdam. </p>

<p>Chargaff, E. 1979. How genetics got a chemical education. <font color="#804040"><em><strong>Ann.
N. Y. Acad. Sci</strong></em></font>. <strong>325</strong>, 345-360. </p>

<p>Crick, F. 1971. General model for the chromosomes of higher organisms. <font color="#804040"><em><strong>Nature</strong></em></font> <strong>234</strong>, 25-27.</p>

<p>Cristillo, A. D., Lillicrap, T. P., Forsdyke, D. R. 1998. Purine-loading of EBNA-1 mRNA
avoids sense-antisense &quot;collisions&quot;. <font color="#804040"><em><strong>FASEB. J</strong></em></font>.
<strong>12</strong>, A1453.</p>

<p>Dalgaard, J.Z., Garrett, A. 1993. Archaeal hyperthermophile genes. In: Kates, M.,
Kushner, D. J., Matheson, A. T. (Eds.), <font color="#804040"><em><strong>The Biochemistry
of Archaea (Archaebacteria)</strong></em></font>. Elsevier, Amsterdam, pp 535-562.</p>

<p>Dang, K. D., Dutt, P. B., Forsdyke, D. R. 1998. Chargaff differences correlate with
transcription direction in the bithorax complex of <i>Drosophila</i>. <font color="#804040"><em><strong>Biochem. Cell Biol</strong></em></font>. <strong>76</strong>,129-137.</p>

<p>Dawkins, R. 1983. Universal Darwinism. In: Bendall, D. S. (Ed.), <font color="#804040"><em><strong>Evolution
from Molecules to Man.</strong></em></font> Cambridge University Press, Cambridge, pp.
405-425.</p>

<p>Eguchi, Y., Itoh, T., Tomizawa, J. 1991. Antisense RNA. <font color="#804040"><em><strong>Annu.
Rev. Biochem</strong></em></font>. 60, 631-652. </p>

<p>Ehrenfeld, E., Hunt, T. 1971. Double-stranded poliovirus RNA inhibits initiation of
protein synthesis by reticulocyte lysates. <font color="#804040"><em><strong>Proc. Natl.
Acad. Sci. USA</strong></em></font> <strong>68</strong>, 1075-1078.</p>

<p>Elson, D., Chargaff, E. 1955. Evidence of common regularities in the composition of
pentose nucleic acids. <font color="#804040"><em><strong>Biochem. Biophys. Acta</strong></em></font>
<strong>17</strong>, 367-376.</p>

<p>Filipski, J. 1990. Evolution of DNA sequences. Contributions of mutational bias and
selection to the origin of chromosomal compartments. <font color="#804040"><em><strong>Adv.
Mut. Res</strong></em></font><i>.</i> <strong>2</strong>:1-54.</p>

<p>Fire, A. 1999. RNA-triggered gene silencing. <font color="#804040"><em><strong>Trends
Genet</strong></em></font>. <strong>15</strong>, 358-363.</p>

<p>Fitch, W. M. 1974. The large extent of putative secondary nucleic acid structure in
random nucleotide sequences of amino acid-derived messenger-RNA. <font color="#804040"><em><strong>J.
Mol. Evol.</strong></em></font> <strong>3</strong>, 279-291.</p>

<p>Forsdyke, D. R. 1995a. A stem-loop &quot;kissing&quot; model for the initiation of
recombination and the origin of introns. <font color="#804040"><strong>Mol. Biol. Evol<i>.</i></strong></font>
<strong>12</strong>, 949-958.</p>

<p>Forsdyke, D. R. 1995b. Conservation of stem-loop potential in introns of snake venom
phospholipase A2 genes. An application of FORS-D analysis. <font color="#804040"><em><strong>Mol.
Biol. Evol</strong></em></font><i>.</i> <strong>12</strong>, 1157-1165.</p>

<p>Forsdyke, D. R. 1995c. Relative roles of primary sequence and (G+C)% in determining the
hierarchy of frequencies of complementary trinucleotide pairs in DNAs of different
species. <font color="#804040"><em><strong>J. Mol. Evol</strong></em></font><i>.</i> <strong>41</strong>,
573-581.</p>

<p>Forsdyke, D. R. 1995d. Entropy-driven protein self-aggregation as the basis for
self/not-self discrimination in the crowded cytosol<i>. </i><font color="#804040"><em><strong>J.
Biol. Sys</strong></em></font>. <strong>3</strong>, 273-287.</p>

<p>Forsdyke, D. R. 1996a. Stem-loop potential in MHC genes: a new way of evaluating
positive Darwinian selection. <font color="#804040"><em><strong>Immunogenetics</strong></em></font>
<strong>43</strong>, 182-189.</p>

<p>Forsdyke, D. R. 1996b. Different biological species &quot;broadcast&quot; their DNAs at
different (C+G)% &quot;wavelengths&quot;. <font color="#804040"><em><strong>J. Theor. Biol</strong></em></font><i>.</i>
<strong>178</strong>, 405-417.</p>

<p>Forsdyke, D. R. 1998. An alternative way of thinking about stem-loops in DNA. A case
study of the human <i>G0S2</i> gene. <font color="#804040"><em><strong>J. Theor. Biol.</strong></em></font>
<strong>192</strong>, 489-504.</p>

<p>Forsdyke, D. R. 1999a. Two levels of information in DNA. Relationship of Romanes'
&quot;intrinsic&quot; variability of the reproductive system, and Bateson's
&quot;residue&quot; to the species-dependent component of the base composition, (C+G)%. <font color="#804040"><em><strong>J. Theor. Biol</strong></em></font>. <strong>201</strong>,
47-61.</p>

<p>Forsdyke, D. R. 1999b. The origin of species, revisited. <font color="#804040"><em><strong>Queen&#146;s
Quarterly</strong></em></font> <strong>106</strong>, 112-134. </p>

<p>Forsdyke, D. R. 2000a. Haldane&#146;s rule: hybrid sterility affects the heterogametic
sex first because sexual differentiation is on the path to species differentiation. <font color="#804040"><em><strong>J. Theor. Biol.</strong></em></font> <strong>204</strong>,
443-452.</p>

<p>Forsdyke, D. R. 2000b. Double-stranded RNA and/or heat-shock as initiators of chaperone
mode switches in diseases associated with protein aggregation. <font color="#804040"><em><strong>Cell
Stress.Chaperones</strong></em></font> <strong>5</strong>, 375-376.</p>

<p>Forsdyke, D. R. 2001. <font color="#804040"><em><strong>Search for a Victorian. The
Origin of Species, Revisited</strong></em></font>. McGill-Queen&#146;s University Press,
Montreal. (in press)</p>

<p>Forterre, P., Elie, C. 1993. Chromosome structure, DNA topoisomerases, and DNA
polymerases in archaebacteria (archeae). In: Kates, M., Kushner, D. J., Matheson, A.
(Eds.), <font color="#804040"><em><strong>The Biochemistry of Archaea (Archaebacteria)</strong></em></font>.
Elsevier, Amsterdam, pp 325-345.</p>

<p>Frank, A. C., Lobry. J. R. 1999. Asymmetric substitution patterns: a review of possible
underlying mutational or selective mechanisms. <font color="#804040"><em><strong>Gene</strong></em></font>.
<strong>238</strong>, 65-77.</p>

<p>Friedman, S. M., Malik, M., Drlica, K., 1995. DNA supercoiling in a thermotolerant
mutant of <i>Escherichia coli</i>. <font color="#804040"><em><strong>Mol. Gen. Genet</strong></em></font>.
<strong>248</strong>, 417-422.</p>

<p>Fulton, A. B. 1982. How crowded is the cytoplasm? <font color="#804040"><em><strong>Cell</strong></em></font>
<strong>30</strong>, 345-347.</p>

<p>Galtier, N., Lobry, J. R. 1997. Relationships between genomic G + C content, RNA
secondary structures, and optimal growth temperature in prokaryotes. <font color="#804040"><em><strong>J.
Mol. Evol</strong></em></font><i>.</i> <strong>44</strong>, 632-636.</p>

<p>Grantham, R. 1972. Codon base randomness and composition drift in coliphage<i>. </i><font color="#804040"><em><strong>Nature New Biol</strong></em></font>. <strong>237</strong>,
265-266.</p>

<p>Grantham, R., Greenland, T., Louail, S., Mouchiroud, D., Prato, J. L., Gouy, M.,
Gautier, C. 1985. Molecular evolution of viruses as seen by nucleic acid sequence study. <font color="#804040"><em><strong>Bull. Inst. Past</strong></em></font>. <strong>83</strong>,
95-148.</p>

<p>Grantham, R., Perrin, P., Mouchiroud, D. 1986. Patterns in codon usage in different
kinds of species<i>. </i><font color="#804040"><em><strong>Oxford Surv. Evol. Biol</strong></em></font>.
<strong>3</strong>, 48-81.</p>

<p>Hamilton, A. J., Baulcombe, D. C. 1999. A species of small antisense RNA in
post-transcriptional gene silencing in plants. <font color="#804040"><em><strong>Science</strong></em></font>
<strong>286</strong>, 950-951.</p>

<p>Hawley, R. S., Arbel, T. 1993.Yeast genetics and the fall of the classical view of
meiosis. <font color="#804040"><em><strong>Cell</strong></em></font><i> </i><strong>72</strong>,
301-303.</p>

<p>Heximer, S. P., Cristillo, A. D., Russell, L., Forsdyke, D. R. 1996. Sequence analysis
and expression in cultured lymphocytes of the human <i>FOSB</i> gene (<i>G0S3</i>).<font color="#804040"><em><strong> DNA Cell Biol</strong></em></font>.<strong>15</strong>,
1025-1038.</p>

<p>Heximer, S. P., Cristillo, A. D., Russell, L., Forsdyke, D. R. 1998. Expression and
processing of <i>G<sub>0</sub>/G<sub>1</sub> Switch Gene 24</i> (<i>G0S24/TIS11/NUP475</i>)
RNA in cultured human blood mononuclear cells. <font color="#804040"><em><strong>DNA Cell
Biol</strong></em></font>. <strong>17</strong>, 249-263.</p>

<p>Isaacs, A. 1962. Antiviral action of interferon. <font color="#804040"><em><strong>Brit.
Med. J</strong></em></font>. <strong>2</strong>, 353-355. </p>

<p>Izant, J. G., Weintraub, H. 1984. Inhibition of thymidine kinase gene expression by
anti-sense RNA: a molecular approach to genetic analysis. <font color="#804040"><em><strong>Cell</strong></em></font>
<strong>36</strong>, 1007- 1015. </p>

<p>Jaeger, J. A., Turner, D. H., Zuker, M. 1990. Predicting optimal and suboptimal
secondary structure for RNA. <font color="#804040"><em><strong>Meth Enzymol</strong></em></font>
<strong>183</strong>, 281-317.</p>

<p>Kleckner, N. 1997. Interactions between and along chromosomes during meiosis<i>. </i><font color="#804040"><em><strong>Harvey Lect</strong></em></font><i>. </i><strong>91</strong>,
21-45.</p>

<p>Lao, P. J., Forsdyke, D. R. 2000. Thermophilic bacteria strictly obey Szybalski's
transcription direction rule and politely purine-load RNAs with both adenine and &nbsp;
guanine<i>. </i><font color="#804040"><em><strong>Genome Res</strong></em></font><i>. </i><strong>10</strong>,
228-236.</p>

<p>Le, S-Y., Maizel, J. V. 1989. A method for assessing the statistical significance of
RNA folding. <font color="#804040"><em><strong>J. Theor. Biol.</strong></em></font> <strong>138</strong>,
495-510.</p>

<p>Lobry, J. R. 1997. Influence of genomic G + C content on average amino-acid composition
of proteins from 59 bacterial species. <font color="#804040"><em><strong>Gene</strong></em></font>
<strong>205</strong>, 309-316.</p>

<p>Majewski, J., Cohan, F. M. 1998. The effect of mismatch repair and heteroduplex
formation on sexual isolation in Bacillus. <strong><font color="#804040"><em>Genetics </em></font>148</strong>,
13-18.</p>

<p>Marcus, P. 1983. Interferon induction by viruses: one molecule of dsRNA as the
threshold for induction. <font color="#804040"><em><strong>Interferon</strong></em></font>
<strong>5</strong>, 115-180.</p>

<p>Mayr, E. 1980. Some thoughts on the history of the evolutionary synthesis. In: Mayr,
E.,Provine, W. B. (Eds.), <font color="#804040"><em><strong>The Evolutionary Synthesis:
Perspectives on the Unification of Biology</strong></em></font>. Harvard Univ. Press,
Cambridge, MA, pp. 1-48.</p>

<p>Melton, D. A. 1985. Injected antisense RNAs specifically block messenger RNA
translation <i>in vivo</i>. <font color="#804040"><em><strong>Proc. Natl. Acad. Sci. USA</strong></em></font>
<strong>82</strong>, 144-148.</p>

<p>Muller, H. J. 1922. Variation due to change in the individual gene. <font color="#804040"><em><strong>Am. Nat</strong></em></font>. <strong>56</strong>, 32-50.</p>

<p>Murchie, A. I. H., Bowater, R., Aboul-Ela, F., Lilley, D. M. J. 1992. Helix opening
transitions in supercoiled DNA. <font color="#804040"><em><strong>Biochem. Biophys. Acta</strong></em></font>
<strong>1131</strong>, 1-15.</p>

<p>Muto, A., Osawa, S. 1987. The guanine and cytosine content of genomic DNA and bacterial
evolution. <font color="#804040"><em><strong>Proc. Natl. Acad. Sci. USA</strong></em></font>
<strong>84</strong>, 166-169.</p>

<p>Nakamura, Y., Gojobori, T., Ikemura, T. 1999. Codon usage tabulated from the
international DNA sequence databases: its status 1999. <font color="#804040"><em><strong>Nucleic
Acids Res</strong></em></font>. <strong>27</strong>, 292.</p>

<p>Naveira, H. F., Maside, X. R. 1998. The genetics of hybrid male sterility in <i>Drosophila</i>.
In: Howard, D. J., Berlocher, S. H. (Eds.), <font color="#804040"><em><strong>Endless
Forms: Species and Speciation</strong></em></font>. Oxford Univ. Press, New York, pp.
330-338.</p>

<p>Oshima, T., Hamasaki, N., Uzawa, T., Friedman, S. M. 1990. Biochemical functions of
unusual polyamines found in the cells of extreme thermophiles. In: Goldembeg, S. H.,
Algranati, I. D. (Eds.)<i>, </i><font color="#804040"><em><strong>The Biology and
Chemistry of Polyamines</strong></em></font>. Oxford University Press, New York, pp 1-10.</p>

<p>Page, A. W., Orr-Weaver, T. L. 1996. Stopping and starting the meiotic cycle<i>. </i><font color="#804040"><em><strong>Curr. Opin. Genet. Devel</strong></em></font>. <strong>7</strong>,
23-31.</p>

<p>Robertson, H. D., Mathews, M. B. 1996. The regulation of the protein kinase PKR by RNA.
<font color="#804040"><em><strong>Biochimie</strong></em></font><i> </i><strong>78</strong>,
909-914. </p>

<p>Rocha, E. P. C., Danchin, A., Viari, A. 1999. Universal replication biases in bacteria<i>.
<font color="#804040"><strong>Mol. Microbiol</strong></font>. </i><strong>32</strong>,
11-16.<i></p>
</i>

<p>Rudner, R., Karkas, J. D., Chargaff, E. 1968. Separation of <i>B. subtilis</i> DNA into
complementary strands. III. <font color="#804040"><em><strong>Proc. Natl. Acad. Sci. USA</strong></em></font>
<strong>60</strong>, 921-922.</p>

<p>Salser, W. 1970. Discussion. <font color="#804040"><em><strong>Cold Spring Harb. Symp.
Quant Biol</strong></em></font>. 35, 19.</p>

<p>Seffens, W., Digby, D. 1999. mRNAs have greater negative folding free energies than
shuffled or codon choice randomized sequences. <font color="#804040"><em><strong>Nucleic
Acids Res</strong></em></font>. <strong>27</strong>, 1578-1584.</p>

<p>Sharp, P. A. 1999. RNAi and double-stranded RNA. <font color="#804040"><em><strong>Genes
Devel</strong></em></font>. <strong>13</strong>, 139-141.</p>

<p>Sharp, P. M., Stenico, M., Peden, J. F., Lloyd, A. T. 1993. Codon usage: mutational
bias, translation selection, or both? <font color="#804040"><em><strong>Biochem. Soc.
Trans</strong></em></font>. <strong>21</strong>, 835-841.</p>

<p>Smithies, O., Engels, W. R., Devereux, J. R., Slightom, J. L., Shen, S. 1981. Base
substitutions, length differences and DNA strand asymmetries in the human G<font FACE="Symbol">l</font> and A<font FACE="Symbol">l</font> fetal globin gene region. <font color="#804040"><em><strong>Cell</strong></em></font> <strong>26</strong>, 345-353. </p>

<p>Stebbins, G. L. 1980. Botany and the synthetic theory of evolution. In: Mayr, E.,
Provine, W. B. (Eds.), <font color="#804040"><em><strong>The Evolutionary Synthesis:
Perspectives on the Unification of Biology</strong></em></font>. Harvard Univ. Press,
Cambridge, MA, pp. 139-152.</p>

<p>Sueoka, N. 1961. Compositional correlations between deoxyribonucleic acid and protein<i>.
<font color="#804040"><strong>Cold Spring Harb. Symp. Quant. Biol</strong></font>. </i><strong>26</strong>,
35-43<i>.</p>
</i>

<p>Sueoka, N. 1995. Intrastrand parity rules of DNA base composition and usage biases of
synonymous codons. <font color="#804040"><strong>J. Mol. Evol<i>.</i></strong></font> <strong>40</strong>,
318-325.</p>

<p>Szybalski, W., Kubinski, H., Sheldrick, P. 1966. Pyrimidine clusters on the
transcribing strands of DNA and their possible role in the initiation of RNA synthesis<i>.
</i>Cold Spring Harbor Symp. <font color="#804040"><em><strong>Quant. Biol</strong></em></font><i>.</i>
<strong>31</strong>, 123-127.</p>

<p>Szybalski, W., Bovre, K., Fiandt, M., Guha, A., Hradecna, Z., Kumar, S., Lozeron, H.
A., Maher, V. M., Nijkamp, H. J. J., Summers, W. C., Taylor, K. 1969. Transcriptional
controls in developing bacteriophages. <font color="#804040"><em><strong>J.Cell Physiol.</strong></em></font>
<strong>74</strong>, suppliment <strong>1</strong>, 33-70.</p>

<p>Tian, B., White, R. J., Xia, T., Welle, S., Turner, D. H., Mathews, M. B., Thornton, C.
A. 2000. Expanded CUG repeat RNAs form hairpins that activate the double-stranded
RNA-dependent protein kinase PKR. <font color="#804040"><em><strong>RNA</strong></em></font>
<strong>6</strong>, 79-87.</p>

<p>Voinnet, O., Pinto, Y. M., Baulcombe, D. C. 1999. Suppression of gene silencing: a
general strategy used by diverse DNA and RNA viruses of plants. <font color="#804040"><em><strong>Proc.
Natl. Acad. Sci. USA</strong></em></font>. <strong>96</strong>, 14147-14152.</p>

<p>Watson, J. D., Crick, F. H. C. 1953. Genetical implications of the structure of
deoxyribonucleic acid. <font color="#804040"><em><strong>Nature</strong></em></font> <strong>171</strong>,
964-967.</p>

<p>Winge, . 1917. The chromosomes, their number and general importance<i>. </i><font color="#804040"><em><strong>Compte. Rend. Trav. Lab. Carlsberg</strong></em></font><i> </i><strong>13</strong>,
131-275.</p>

<p><img src="images/colorbar.gif" alt="colorbar.gif (2362 bytes)" WIDTH="596" HEIGHT="6"></p>

<p><big>For Chargaff's &quot;<font color="#800040"><strong>In Praise of Smallness</strong></font>&quot;
<a href="peerrev0.htm#IN PRAISE OF SMALLNESS">(Click Here) </a></big></p>

<p><big>For Chargaff's &quot;<font color="#800040"><strong>How Genetics Got a Chemical
Education</strong></font>&quot;&nbsp; <a href="bioinfo1.htm#How Genetics Got a Chemical Education">(Click Here)</a></big></p>

<p><big>Go to: <font color="#800040"><strong>Bioinformatics Index</strong></font> &nbsp; <a href="bioinfor.htm#Bioinformatics (Genomics)">(Click Here)</a></big></p>

<p><big>Go to: <font color="#800040"><strong>Homepage</strong></font> <a href="homepage.htm#Homepage">(Click Here)</a></big></p>

<p><img src="images/colorbar.gif" alt="colorbar.gif (2362 bytes)" WIDTH="596" HEIGHT="6"></p>

<p><small>This page was last edited on <!--webbot bot="Timestamp" S-Type="EDITED" S-Format="%d %b %Y" startspan -->10 Dec 2009<!--webbot bot="Timestamp" endspan i-checksum="14421" --> by Donald Forsdyke</small></p>
</body>
</html>
