<BASE HREF="http://www.molfunction.com/">
<html>

<head>
<title>Institute of Molecular Function</title>
<meta name="copyright" content="Copyright (C) INSTITUTE OF MOLECULAR FUNCTION">
<meta name="Author" content="Institute of Molecular Function">
<meta name="robots" content="index,follow">
<meta name="keywords" content="Institute of Molecular Function,Homology,Modeling,HyperChem,Docking Study,Simulation,in silico,Gaussian,Interface,PIEFII,Interaction,Site,Prediction,Complex,Formation,Scaffold,Pharmacophore,Molecule,Function,Design,Drug,Flexible,Methodology,Simulation,ligand,Protein,Biomacromolecule,NAMD,VMD,ABINIT-MP,GAMESS,BioStatio,Viewer,Fu,Facio,AutoDock,Vina,Fragment,Molecular Orbital,Quantum,Calculation,Molecular Dynamics">
<meta name="description" content="HyperChem can be upgraded as the latest drug discovery system by the Homology Modeling Professional for HyperChem and Docking Study with HyperChem applications. The latest in silico drug design is supported by the ab-initio biomacromolecule-ligand interaction site and potential ligand prediction technology. Many types of docking algorithm can be used for the biomacromolecule and ligand docking simulations. Molecular modeling such as protein homology modeling and nucleic acid modeling become easily available.">
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<link rel="shortcut icon" href="molfunc.ico">
<style type="text/css">
.style1 {
	color: #0000FF;
	font-weight: bold;
}
.style7 {
	color: #FF0000;
}
.style10 {
	font-family: "Lucida Sans Unicode";
}
.style11 {
	font-weight: bold;
	font-family: "Times New Roman", Times, serif;
	font-size: small;
}
.style13 {
	border-width: 0px;
}
.style18 {
	color: #000000;
}
.style23 {
	font-family: "Times New Roman", Times, serif;
	font-style: italic;
}
.style24 {
	color: #000000;
	font-family: "Times New Roman", Times, serif;
}
.style27 {
	color: #FF0000;
	font-family: "Times New Roman", Times, serif;
	font-size: small;
}
.style31 {
	font-family: "Times New Roman", Times, serif;
}
.style32 {
	font-family: "Times New Roman", Times, serif;
	font-size: small;
}
.style33 {
	color: #000000;
	font-family: "Times New Roman", Times, serif;
	font-size: small;
}
.style34 {
	font-size: small;
}
.style35 {
	color: #FF0000;
	font-size: small;
}
.style36 {
	font-size: medium;
	color: #FF0000;
}
.style38 {
	color: #000080;
}
.style39 {
	color: #FF0000;
	font-weight: bold;
}
.style42 {
	font-size: large;
}
.style43 {
	color: #FF0000;
	font-size: x-small;
}
</style>
<meta name="Microsoft Border" content="tlb, default">
</head>

<body><!--msnavigation--><table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td>
<p align="center"><a href="index.htm"><img alt="Institute of Molecular Function" src="images/top.jpg" width="1050" height="356" class="style1"></a></p>
</td></tr><!--msnavigation--></table><!--msnavigation--><table dir="ltr" border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td valign="top" width="1%">
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><map name="FPMap0_I2" id="FPMap0">
<area href="price.htm" shape="rect" coords="0, 116, 122, 158">
<area href="order.htm" shape="rect" coords="0, 158, 121, 199">
<area coords="0, 199, 121, 241" shape="rect" href="download.htm">
<area href="regist.htm" shape="rect" coords="0, 282, 121, 324">
<area href="index.htm" shape="rect" coords="0, 2, 121, 37">
<area href="products.htm" shape="rect" coords="0, 37, 122, 76">
<area coords="0, 76, 121, 116" shape="rect" href="tutorial.htm">
<area href="support.htm" shape="rect" coords="0, 241, 122, 282">
<area href="FAQ.htm" shape="rect" coords="0, 324, 122, 365">
<area href="links.htm" shape="rect" coords="0, 365, 122, 405">
<area href="profile.htm" shape="rect" coords="0, 405, 122, 450">
</map>
<img border="0" src="images/left.gif" width="132" height="454" usemap="#FPMap0_I2"> 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
<p><b><font size="3"><i>&nbsp;&nbsp; </i></font><a href="jp/index.htm">
<font size="3"><i>Japanese</i></font></a></b></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><a href="https://www.youtube.com/channel/UCjZxRYJVvE-Nus45s0I82GA">
<img alt="" src="images/youtube_social_squircle_red.png" width="50" height="50"></a>
<a href="https://www.facebook.com/&#20998;&#23376;&#27231;&#33021;&#30740;&#31350;&#25152;-1656468537909317/">
<img alt="" src="images/f-ogo_RGB_HEX-58.png" width="48" height="48"></a></p>
<p>&nbsp;</p>
<p><i><font size="2">&nbsp;&nbsp; Last Modified</font></i></p>
<p><i><font size="2">&nbsp;&nbsp; 8 February 2019</font></i></p>
</td><td valign="top" width="24"></td><!--msnavigation--><td valign="top">
<p><map name="FPMap1">
<area href="molecule.htm" shape="rect" coords="0, 0, 218, 46" alt="Theoretical Organic Chemistry">
<area href="protein.htm" shape="rect" coords="218, 0, 443, 46" alt="Nuclear Receptor Chemistry">
<area href="bioinformatics.htm" shape="rect" coords="443, 0, 642, 46" alt="Structural Bioinformatics">
<area href="molfunctech.htm" shape="rect" coords="642, 0, 841, 46" alt="Rational Drug Design">
</map><img border="0" src="images/category.gif" width="842" height="47" usemap="#FPMap1"></p>
<p><a href="software1.htm"><img border="2" src="images/logo.gif" width="130" height="55" alt="Homology Modeling for HyperChem Logo"></a>
<a href="software4.htm"><img border="2" src="images/logo3.gif" width="130" height="55" alt="Gaussian Interface for HyperChem Logo"></a>
<a href="software6.htm"><img border="2" src="images/logo5.gif" width="130" height="55" alt="Full-Auto Interactive, Muti-layer ONIOM Interface Logo"></a>
<a href="software5.htm"><img border="2" src="images/logo4.gif" width="130" height="55" alt="Docking Study with HyperChem Logo"></a>
<a href="software8.htm"><img border="2" src="images/logo7.gif" width="130" height="55" alt="Mol Dimension Logo"></a>
<a href="technology.htm"><img border="2" src="images/logo6.gif" width="130" height="55" alt="Virtual Screening System Technology"></a>
<a href="software10.htm"><img border="2" src="images/logo10.jpg" width="130" height="55" alt="AutoDock Vina In Silico Screenings Interface"></a>
<a href="software9.htm"><img border="2" src="images/logo8.gif" width="130" height="55" alt="The Next-generation Drug Design Technology"></a>
<a href="../jp/drugdesign.htm"><img border="2" src="images/logo9.gif" width="130" height="55" alt="Drug Design Methodology for Organic Chemists"></a>
<a href="../jp/MFDD.htm"><img border="2" src="images/collaboration.jpg" width="130" height="55" alt="MFDD In Silico Drug Design Research Service"></a>
<a href="hyperchem.htm"><img border="2" src="images/HyperChem_logo.png" width="130" height="55" alt="HyperChem"></a></p>
<p><a href="../jp/MFDD.htm"><img alt="MFDD In Silico Drug Design Service" src="images/MFDD/MFDD.jpg" width="800" height="400" class="style13"></a></p>
<p>&nbsp;</p>
<hr>
<p><a href="announce201802.htm"><img alt="Structure-based Drug Design System" src="images/MFDD/SBDD_top2.jpg" width="800" height="400" class="style13"></a></p>
<p>&nbsp;</p>
<hr>
<p><a href="software10.htm"><img alt="AutoDock Vina In Silico Screenings Interface" src="images/AutoDockVina/AutoDock%20Vina%20Tutorial.JPG" width="800" height="407" class="style13"></a></p>
<p>&nbsp;</p>
<hr>
<p><a href="molfunctech.htm"><img alt="The latest in silico technique of Institute of Molecular Function" src="images/MF%20in-silico%20technology.png" width="800" height="600" class="style13"></a></p>
<p>&nbsp;</p>
<hr>
<p><a href="hyperchem.htm"><img alt="HyperChem Release 8 for Windows" src="images/hyperchem/MF-HyperChem8-2.jpg" width="800" height="449" class="style13"></a></p>
<p>&nbsp;</p>
<hr>
<p class="style42"><strong><em>HyperChem</em><span class="style42"> can be drastically upgraded as the latest drug discovery system by the <em>Homology Modeling Professional for HyperChem</em> (HMHC) and <em>Docking Study with HyperChem</em> (DSHC) applications.</span></strong></p>
<p>&nbsp;</p>
<p><b><font size="4" color="#FF0000">Announce</font></b></p>
<p><b><font size="5" color="#0000FF"><i><a href="announce201802.htm">What's New in Revision H1.</a></i></font></b></p>
<p class="style36"><strong><em>The Latest In Silico Drug Design Platform: Evolution of Docking Study with HyperChem, Revision H1 and Homology Modeling Professional for HyperChem, Revision H1.</em></strong></p>
<p class="style7">Strengthened quantum mechanics calculation program, <em>Gaussian</em>,<em>ONIOM Interface</em>, compatible to file format of molecular dynamics calculation program, <em>NAMD</em> (<em>VMD</em>: <em>CHARMM</em>-based PDB), compatible to file format of fragment molecular orbital calculation programs, <em>ABINIT-MP</em> (<em>BioStation Viewer</em>) and <em>GAMESS</em> (<em>Fu</em> /<em>Facio</em>), and supported <em>AutoDock Vina Virtual Screening Interface</em> (preparation of compound database - preparation of all input files - execution - viewing and filtering of hits; supports also simultaneous analysis of both <em>Docking Study</em> and <em>AutoDock Vina</em> results)</p>
<p>&nbsp;</p>
<p class="style38">In quantum mechanics calculations for the entire structure of big molecular system such as biomacromolecular system, the single point calculations for the initial structure obtained from classical molecular mechanics calculations and/or classical molecular dynamics calculations cannot converge or will give abnormal energies. For obtaining reliable results from quantum mechanics calculations such as fragment molecular orbital methods of entire molecular system, the initial structure must be prepared using geometrical optimization calculations via ONIOM methods. <em>ONIOM Interface for Receptor</em> can provide the best solution for the quantum mechanics calculations of the entire molecular system of which the precise initial structure is prepared from <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em>.</p>
<p><span class="style38"><strong>Motonori Tsuji, et. al. FEBS Open Bio., 7, 391-396, 2017, </strong></span> <a href="http://onlinelibrary.wiley.com/doi/10.1002/2211-5463.12188/full"><strong><span class="style38">DOI: 10.1002/2211-5463.12188</span></strong></a>.</p>
<p>&nbsp;</p>
<hr>
<p><b><i><font color="#000080" size="5"><a href="software5.htm"><font color="#000080">Docking Study with HyperChem</font></a></font></i><font size="5"><font color="#000080">, </font><font color="#FF0000">Revision H1</font></font></b></p>
<p class="style39"><font size="4"><em>with <a href="software10.htm"><span class="style7">AutoDock Vina In Silico Screenings Interface</span></a></em></font></p>
<p class="style39"><em><font size="4">Compatible to NAMD Molecular Dynamics and ABINIT-MP / GAMESS Fragment Molecular Orbital Quantum Mechanics Calculations for Entire Molecular System</font></em></p>
<p><b><font color="#0000FF" size="4">- Carrying The Next-generation Drug Discovery Technology, The PIEFII Technology -</font></b></p>
<p>&nbsp;</p>
<p><font color="#FF0000" size="4"><i><b>Powered by Docking Study with HyperChem</b></i></font>&nbsp;&nbsp; <font color="#FF0000"><b>THE IMPOSSIBLE BECOMES POSSIBLE.</b></font></p>
<p><b>Advanced Technology &amp; The High-Performance, Biomacromolecule- &amp; Ligand-Flexible Docking &amp; Screening Program Package</b></p>
<p><em>Docking Study with HyperChem</em> supports the comprehensive and automatic biomacromolecule (protein and nucleic acid molecule systems)- and ligand (small molecule and peptide)-flexible docking simulations, and thus supports the drug design and the in silico screening of a drug candidate. </p>
<p><img alt="Biomacromolecule- &amp; Ligand-Flexible Docking" src="images/dockingstudy.gif" width="480" height="360"></p>
<p>&nbsp;</p>
<hr>
<p><font size="5"><b><i><font color="#000080"><a href="software1.htm"><font color="#000080">Homology Modeling Professional for HyperChem</font></a></font></i>, </b><font color="#FF0000"><b> Revision H1</b></font></font></p>
<p><font color="#FF0000" size="4"><b>with <i><a href="software4.htm"><font color="#FF0000">Gaussian Interface for HyperChem</font></a> &amp; <a href="software6.htm"><font color="#FF0000">ONIOM Interface for Receptor</font></a></i></b></font></p>
<p class="style7"><em><font size="4"><strong>Compatible to NAMD Molecular Dynamics and ABINIT-MP / GAMESS Fragment Molecular Orbital Quantum Mechanics Calculations for Entire Molecular System</strong></font></em></p>
<p>&nbsp;</p>
<p><font color="#FF0000" size="4"><i><b>Powered by Homology Modeling Professional for HyperChem</b></i></font>&nbsp;&nbsp; <font color="#FF0000"><b>THE IMPOSSIBLE BECOMES POSSIBLE.</b></font></p>
<p><b>New, High-Performance Software for Biomacromolecular System Modeling, Functional Analysis, and Simulation.</b></p>
<p><i>Homology Modeling Professional for HyperChem</i> is the latest molecular modeling package which can carry out the molecular modeling, functional analysis, and simulations of a big molecular system using comprehensively the ab-initio quantum chemistry calculations and the molecular dynamics simulations as well as the molecular mechanics calculations. The program can control seamlessly both <i>HyperChem</i> and <i>Gaussian</i> packages and can be simultaneously fed back their calculation results to the big molecular system in the <i>HyperChem</i> graphical user interface. The whole molecular system can be operated in a quantum chemistry via the full-automatic <i>ONIOM</i> interface to <i>Gaussian</i>. All operations are fully automated in the GUI-based manner, and thus the program supports the logical molecular modeling, functional analysis, and simulations in the life science research such as the structure-based drug design, as well as an unexplored research. </p>
<p>&nbsp;</p>
<p><img alt="Homology Modeling Module Program" src="images/HM-F1-HM.gif" width="480" height="360"></p>
<p>&nbsp;</p>
<hr>
<p>&nbsp;</p>
<hr color="#000000" width="60%" size="10" align="left">
<p align="left"><font color="#FF0000" size="4"><b>The Next-generation Drug Discovery Technology</b></font></p>
<p align="left"><font size="5" color="#000080"><i><b><a href="software9.htm"><font color="#000080">PEIFII</font></a></b></i><sup><span class="style43">Patent Right</span></sup></font></p>
<p align="left"><b><font color="#0000FF"><em>Ab-Initio</em> Biomacromolecule-Ligand Interaction Site </font></b></p>
<p align="left" class="style1">Structure-based Pharmacophore Prediction Technology</p>
<p align="left"><b><font color="#0000FF">Potential Ligand Prediction Technology</font></b></p>
<p align="left"><font color="#0000FF"><a href="software9.htm">A New Drug Design Methodology</a></font></p>
<p><img border="0" src="images/piefii1.gif" width="384" height="205"></p>
<p><img border="0" src="images/piefii2.gif" width="467" height="218"></p>
<hr color="#000000" width="60%" size="10" align="left">
<p>&nbsp;</p>
<hr>
<p>&nbsp;</p>
<p><img alt="In Silico Drug Design Flow" src="images/MF%20in%20silico%20screening%20flow.png" width="600" height="450"> <img alt="in silico drug design system" src="images/sbdd/SBDD.gif" width="600" height="450"></p>
<p>&nbsp;</p>
<p><span class="style11">Latest News&nbsp;&nbsp; </span><span class="style32"><a href="news.htm">Archives</a></span></p>
<hr>
<p class="style33">2019/02/08&nbsp;&nbsp; Revised the home pages for mobile devices.</p>
<p class="style33">2019/01/26&nbsp;&nbsp; Open the MFDD Service Web Site (Japanese)</p>
<p class="style33">2018/07/01&nbsp;&nbsp;&nbsp;15th Anniversary</p>
<p class="style33">2018/06/07-08&nbsp;&nbsp; Motonori Tsuji will speak a research entitled &quot;<a href="RX02.pdf">Advancement of the Current Drug Design Technologies via <em>In Silico</em> Drug Design System,<em> Docking Study with HyperChem</em> (DSHC) and <em>Homology Modeling Professional for HyperChem</em> (HMHC).</a>&quot; at the 2018 Spring annual meeting of Society of Computer Chemistry, Japan.</p>
<p class="style33"><img alt="In Silico Drug Design System" src="images/Society/SCCJ2018.jpg" width="320" height="300"></p>
<p class="style33">2018/03/27&nbsp;&nbsp; Motonori Tsuji will speak a research entitled &quot;Antagonist-Perturbation Mechanism for Activation Function-2 Fixed Motifs: Active Conformation and Docking Mode of Retinoid X Receptor Antagonists&quot; at the 138th annual meeting of the pharmaceutical society of Japan.</p>
<p class="style33"><img alt="AF-2 Fixed Motif Perturbation Mechanism" src="images/Society/JP138.jpg" width="184" height="300"></p>
<p class="style33">2018/03/21&nbsp;&nbsp; Sophisticated Molecular Modeling Software, <em>HyperChem</em> site is started in Japan.</p>
<p class="style33"><span class="style34">2018/02/23&nbsp;&nbsp; </span><font size="2" class="style32"><font color="#FF0000"><b> Announcing&nbsp; <em>Docking Study with HyperChem</em>, Revision H1 and </b></font><b><font color="#FF0000"><em>Homology Modeling Professional for HyperChem</em>, Revision H1</font></b><font color="#FF0000"><b>.</b></font></font></p>
<p class="style32">2018/01/09&nbsp;&nbsp; <span class="style7"><strong>The Latest <em>In Silico</em> Drug Design Platform: Evolution of <em>Docking Study with HyperChem</em>, Revision H1 and <em>Homology Modeling Professional for HyperChem</em>, Revision H1. Coming Soon. (Strengthened quantum mechanics calculation program, <em>Gaussian</em>, <em>ONIOM Interface</em>, compatible to file format of molecular dynamics calculation program, <em>NAMD</em> (<em>VMD</em>: <em>CHARMM</em>-based PDB), compatible to file format of fragment molecular orbital calculation programs, <em>ABINIT-MP</em> (<em>BioStation Viewer</em>) and <em>GAMESS</em> (<em>Fu</em> /<em>Facio</em>), and supported <em>AutoDock Vina</em> Virtual Screening Interface (preparation of compound database - preparation of all input files - execution - viewing and filtering of hits; supports also simultaneous analysis of both Docking Study and <em>AutoDock Vina</em> results))</strong></span></p>
<p class="style33">2017/11/19&nbsp;&nbsp; Motonori Tsuji spoke a research entitled &quot;Identifying the Receptor Subtype Selectivity of Retinoids via Docking, QM/MM, and Quantum Mechanics Calculations: ATRA Can Act as an Endogenous RXR Ligand&quot; at the 28th symposium of retinoids of Japan society for retinoid research.</p>
<p class="style24"><span class="style34">2017/05/16&nbsp;&nbsp; </span> <span class="style7"><strong><span class="style34">Origin of the agonism and antagonism mechanisms in the nuclear receptors via quantum mechanics (using core technologies of <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em>) has been accepted for publication. </span> </strong></span><span class="style35"><strong>Motonori Tsuji. <em>J. Comput. Aided Mol. Des.</em> 31, 577-585, 2017, DOI: 10.1007/s10822-017-0025-6.</strong></span></p>
<p class="style24"><span class="style34">2017/04/16&nbsp;&nbsp; </span> <strong><span class="style34">Confirmed that the products are compatible to <em>Gaussian16W</em> 64-bit multi-processor version (Revision A.03). I really appreciate great help of Gaussian, Inc.</span></strong></p>
<p class="style24"><span class="style34">2017/03/28&nbsp;&nbsp; </span> <span class="style35"><strong>All series of </strong></span><em><a href="hyperchem.htm"><strong><span class="style35">HyperChem</span></strong></a></em><span class="style35"><strong> of Hypercube, Inc. are available from our web site.</strong></span></p>
<p class="style32">2017/03/27&nbsp;&nbsp; Motonori Tsuji spoke a research entitled &quot;Helix H3 Three-Point Initial-Binding Hypothesis: Understanding of Both the Ligand-Trapping Mechanism and the Simultaneous Structural Transition from Apo-Form to Holo-Form of the Ligand-Binding Domain in the Nuclear Receptor Superfamily&quot; at the 137th annual meeting of the pharmaceutical society of Japan.</p>
<p class="style32"><img alt="Helix H3 Three-Point Initial-Binding Hypothesis" src="images/Society/JP137.jpg" width="166" height="300"></p>
<p class="style31"><span class="style34">2017/02/15&nbsp;&nbsp; </span> <span class="style7"><strong><span class="style34"><a href="PressRelease170215.pdf">Press Release</a>: Origin of the receptor subtype selectivity via quantum mechanics has been published. Motonori Tsuji, et. al. </span> <em><span class="style34">FEBS Open Bio.</span></em><span class="style34"> </span></strong></span><span class="style35"> <strong>7, 391-396, </strong> </span> <span class="style7"><strong><span class="style34"> 2017, DOI: 10.1002/2211-5463.12188.</span></strong></span></p>
<p class="style31"><span class="style34">2016/12/29&nbsp;&nbsp; </span> <span class="style35"><strong>Origin of the receptor subtype selectivity via quantum mechanics has been accepted for publication.</strong></span></p>
<p class="style31"><span class="style34">2016/04/16&nbsp;&nbsp; </span> <strong><span class="style34">Confirmed that <em>ONIOM Interface for Receptor</em> and <em>Gaussian Interface for HyperChem</em> were compatible with the latest <em>Gaussian09W</em> (64bit-multiprocessor version, Revision E.01) under support of Gaussian, Inc. I really appreciate great help of Gaussian, Inc.</span></strong></p>
<p class="style32">2016/03/27&nbsp;&nbsp; Motonori Tsuji spoke a research entitled &quot;Geometrical Dependence of the Highest Occupied Molecular Orbital in Bicyclic Systems: Origin of the Facial Stereoselectivity of Bicyclic Olefins&quot; at the 136th annual meeting of the pharmaceutical society of Japan.</p>
<p class="style32"><span class="style34">2016/03/26&nbsp;&nbsp; </span><font size="2" class="style32"><font color="#FF0000"><b> Announcing&nbsp; <em>Docking Study with HyperChem</em> 2016 and </b></font> <b><font color="#FF0000"><em>Homology Modeling Professional for HyperChem</em></font></b><font color="#FF0000"><b> 2016, Revision G1.</b></font></font></p>
<p class="style31"><span class="style34">2016/03/26&nbsp;&nbsp; </span> <strong><span class="style34">The function of </span> </strong> <a href="software6.htm"><strong><span class="style34"><em>ONIOM Interface for Receptor</em></span></strong></a><strong><span class="style34"> was strengthened (compatible to whole system of protein and nucleic acid molecular system).</span></strong></p>
<p class="style32">2015/12/01&nbsp;&nbsp; Confirmed that <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em> were compatible with Windows 10 Professional (64 bit Version).</p>
<p><span class="style32">2015/11/04&nbsp;&nbsp; </span> <span class="style7"><span class="style32"><strong><a href="PressRelease151104.pdf">Press Release</a>: Origin of the ligand recognition mechanism in the nuclear receptor superfamily has been published. Motonori Tsuji.</strong></span> <span class="style31"> <em><strong><span class="style34">J. Mol. Graph. Model.</span></strong></em><strong><span class="style34"> 62, 262-275, 2015.</span></strong></span></span></p>
<p class="style32">2015/10/07&nbsp;&nbsp; The first success of the structural transition of the holo-form formation simulations and the discovery of the driving forces involved in the ligand entry in the nuclear receptor superfamily using <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em> has been accepted for publication.</p>
<p class="style31"><span class="style34">2015/09/13 </span> <span class="style7"><strong><span class="style34">&nbsp; A proposed molecular mechanism of ATRA via retinoid X receptors (using core technologies of <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em>) has been accepted for publication. Motonori Tsuji, et. al.</span> <em><span class="style34">J. Comput. Aided Mol. Des.</span></em><span class="style34"> 29, 975-988, 2015.</span></strong></span></p>
<p><span class="style32">2015/04/17 </span> <span class="style7"><strong><span class="style31"><span class="style34">&nbsp; <a href="PressRelease150417.pdf">Press Release</a>: Origin of the facial stereoselectivity, which is the most important and unsolved issue in organic chemistry, has recently been accepted for publication. Motonori Tsuji.</span> <em><span class="style34">Asian J. Org. Chem.</span></em><span class="style34"> 4, 659-673, 2015.</span></span></strong></span></p>
<p class="style32">2015/03/26&nbsp;&nbsp; Motonori Tsuji spoke a research entitled &quot;Structure and function of the local motifs in the canonical structure of the ligand-binding domain in the nuclear receptor superfamily&quot; at the 135th annual meeting of the pharmaceutical society of Japan.</p>
<p class="style10"><span class="style32">2015/01/04&nbsp;&nbsp;</span><font size="2" class="style32"><font color="#FF0000"><b> Announcing&nbsp; <em>Docking Study with HyperChem</em> 2015 and </b></font> <b><font color="#FF0000"><em>Homology Modeling Professional for HyperChem</em></font></b><font color="#FF0000"><b> 2015, Revision G1.</b></font></font></p>
<p><font size="2" class="style32">2014/10/14&nbsp;&nbsp;<font color="#FF0000"><b> Announcing&nbsp; <em>Docking Study with HyperChem</em> and </b></font> <b><font color="#FF0000"><em>Homology Modeling Professional for HyperChem</em></font></b><font color="#FF0000"><b>, Revision F3.</b></font></font></p>
<p><font size="2" class="style32">2014/10/02&nbsp;&nbsp; Confirmed that <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em> were compatible with Windows 10 Technical Preview (64 bit Version).</font></p>
<p><span class="style32">2014/03/18&nbsp;&nbsp; </span><span class="style27"><strong>A paper regarding the folding mechanism and the agonism and atagonism mechanism of the ligand-binding domain in the nuclear receptor superfamily has recently been published. Motonori Tsuji </strong></span><em><span class="style27"><strong>J. Struct. Biol.</strong></span></em><span class="style27"><strong> 185, 355-365, 2014.</strong></span></p>
<hr>
<p>&nbsp;</p>
<p class="style11">Topics</p><hr>
<p class="style33">2018/07/01&nbsp;&nbsp;&nbsp; 15th Anniversary</p>
<p class="style33">2017/05/16&nbsp;&nbsp; <span class="style18">Origin of the agonism and antagonism mechanisms in the nuclear receptors via quantum mechanics has been accepted for publication. </span><span class="style34">Motonori Tsuji. </span> <span class="style18"> <em><span class="style34">J. Comput. Aided Mol. Des.</span></em><span class="style34"> 31, 577-585, </span></span><span class="style34">2017, in press, DOI: 10.1007/s10822-017-0025-6.</span></p>
<p class="style33">2017/04/16&nbsp;&nbsp; Confirmed that the products are compatible to <em>Gaussian16W</em> 64-bit multi-processor version (Revision A.03). I really appreciate great help of Gaussian, Inc.</p>
<p class="style24"><span class="style34">2017/03/28&nbsp;&nbsp; All series of </span> <em><a href="hyperchem.htm"><span class="style34">HyperChem</span></a></em><span class="style34"> of Hypercube, Inc. are available from our web site.</span></p>
<p class="style24"><span class="style34">2016/12/29&nbsp;&nbsp; Origin of the receptor subtype selectivity via quantum mechanics has been accepted for publication. Motonori Tsuji, et. al., </span> <em><span class="style34">FEBS Open Bio.</span></em><span class="style34"> 7, 391-396, 2017, </span> <a href="http://onlinelibrary.wiley.com/doi/10.1002/2211-5463.12188/full"><span class="style34">DOI: 10.1002/2211-5463.12188</span></a><span class="style34">.</span></p>
<p class="style33">2016/04/16&nbsp;&nbsp; Confirmed that the products are compatible to <em>Gaussian09W</em> 64-bit multi-processor version (Revision E.01). I really appreciate great help of Gaussian, Inc.</p>
<p class="style18"><span class="style32">2015/10/07&nbsp;&nbsp; </span> <span class="style18"><span class="style31"><span class="style34">Origin of the ligand recognition mechanism in the nuclear receptor superfamily has been published</span><em><span class="style34">. </span> </em> <span class="style34">Motonori Tsuji. </span> <em><span class="style34">J. Mol. Graph. Model.</span></em><span class="style34"> 62, 262-275, 2015.</span></span></span></p>
<p class="style18">
<img alt="Ligand Entry to Nuclear Receptor" src="images/HelixH3_Initial-Binding_Hypothesis.gif" width="500" height="255"><img alt="Ligand Entry to Nuclear Receptor" src="images/HelixH3_Initial-Binding_Hypothesis2.gif" width="500" height="255"></p>
<p class="style24"><span class="style34">2015/09/13&nbsp;&nbsp; A proposed molecular mechanism of ATRA via retinoid X receptor (using core technologies of <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em>) has been accepted for publication. Motonori Tsuji, et. al. </span> <em><span class="style34">J. Comput. Aided Mol. Des</span></em><span class="style34">. 29, 975-988, 2015.</span></p>
<p class="style24"><span class="style34">2015/04/17&nbsp;&nbsp; Discovery of the underlying rule for the facial stereoselectivity, which is the most important and unsolved issue in organic chemistry, has recently been accepted for publication. Motonori Tsuji.</span><em><span class="style34"> Asian J. Org. Chem.</span></em><span class="style34"> 4, 659-673, 2015</span><strong><span class="style34">.</span></strong></p>
<p class="style33"><img alt="Tsuji Model for Facial Stereoselectivity" src="images/organic/TSUJI_model_Facial_Stereoselectivity.GIF" width="533" height="400"></p>
<p class="style24"><span class="style34">2014/03/18&nbsp;&nbsp; A paper regarding the folding mechanism and the agonism and atagonism mechanism of the ligand-binding domain in the nuclear receptor superfamily has recently been published. Motonori Tsuji. </span> <em><span class="style34">J. Struct. Biol.</span></em><span class="style34"> 185, 355-365, 2014.</span></p>
<p class="style32"><a href="protein.htm"><img alt="The folding mechanism and the agonism and atagonism mechanism of the ligand-binding domain in the nuclear receptor superfamily" src="images/jsb_ga.jpg" width="500" height="206" class="style13"></a></p>
<p class="style32">&nbsp;</p>
<p class="style32">2013/07/01&nbsp;&nbsp;&nbsp; 10th Anniversary</p>
<p class="style32">2012/11/30&nbsp;&nbsp;&nbsp; Acquired patent rights regarding structural-based pharmacophre prediction technology.</p>
<p class="style10"><span class="style32">2012/07&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><font size="2" class="style32"> <em>Homology Modeling Professional for HyperChem</em> was cited in an article.<em>Plant Cell Physiol.</em> 2012 Jul; 53 (9): 1638-1647.</font></p>
<p><font size="2" class="style34"><span class="style31">2010/12&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <em>Homology Modeling Professional for HyperChem</em> was cited in an article.</span><span class="style23">Biochemistry</span><span class="style31"> 2010 Dec; 49 (50): 10647-10655.</span></font></p>
<p><font size="2" class="style34"><span class="style31">2010/04/30&nbsp;&nbsp;&nbsp; <em>Homology Modeling Professional for HyperChem</em> was cited in an article.</span><span class="style23">Biochemical and Biophysical Research Communications</span><span class="style31"> 2010 Apr 30; 339 (2): 173-177.</span></font></p>
<p><font size="2" class="style34"><span class="style31">2008/02&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em> were cited in an article.</span><span class="style23"> Science</span><span class="style31"> 2008 Feb 1; 319 (5863): 624-7.</span></font></p>
<p class="style32">2008/01/11&nbsp;&nbsp;&nbsp; Motonori Tsuji was invited as a guest speaker at Business Research Institute.</p>
<p class="style32">2007/11/15&nbsp;&nbsp;&nbsp; <em><a href="hyperchem.htm">HyperChem</a></em> of Hypercube, Inc. is available as bundle with my products.</p>
<p><font size="2" class="style34"><span class="style31">2007/06/29&nbsp;&nbsp;&nbsp; <em>Docking Study with HyperChem</em> and <em>Homology Modeling Professional for HyperChem</em> were introduced in an article. Motonori Tsuji. </span><span class="style23">Molecular Science</span><span class="style31"> 2007; 1: NP004.</span></font></p>
<p class="style32">2007/06/16&nbsp;&nbsp;&nbsp; I really appreciate great cooperation of Neil S. Ostlund, President and Chief Executive Officer of Hypercube, Inc.</p>
<p class="style32">2006/06/05&nbsp;&nbsp;&nbsp; <em>Docking Study with HyperChem</em> is available.</p>
<p class="style32">2005/12/01&nbsp;&nbsp;&nbsp; <em>Homology Modeling Professional for HyperChem</em> is available.</p>
<p class="style32">2005/11/28&nbsp;&nbsp;&nbsp; <em>Gaussian Interface for HyperChem</em> and <em>ONIOM Interface for Receptor</em> are available.</p>
<p class="style32">2005/08/09&nbsp;&nbsp;&nbsp; My products have been linked to the Gaussian Official site.</p>
<p class="style32">2005/07/11&nbsp;&nbsp;&nbsp; <em>Homology Modeling for HyperChem</em> is available.</p>
<p class="style32">2005/05/01&nbsp;&nbsp;&nbsp; Institute of Molecular Function Web site is started.</p>
<p class="style32">2003/08/06&nbsp;&nbsp;&nbsp; A paper regarding the electronic structure of carboranes and the most stable conformation of cyclopropane ring has been accepted for publication. J. Org. Chem. 69, 4063-4074, 2004.</p>
<p class="style32">2003/08/06&nbsp;&nbsp;&nbsp; A paper regarding carboranyl carbocation has been accepted for publication. J. Org. Chem. 68, 9589-9597, 2003.</p>
<p class="style32">2003/07/01&nbsp;&nbsp;&nbsp; Institute of Molecular Function has been established by Motonori Tsuji.</p>
<hr>
<p class="style32">&nbsp;</p>
<p><font size="2"><sup>*</sup> <i> HyperChem</i> is a registered trademark of Hypercube, Inc.</font></p>
<p><font size="2"><sup>*&quot;</sup> <i>Gaussian</i> is a registered trademark of Gaussian, Inc.</font></p>
<p>&nbsp;</p>

<!--msnavigation--></td></tr><!--msnavigation--></table><!--msnavigation--><table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td>
<p align="center"><font size="2"><a href="sitemap.htm">Site Map</a>&nbsp;<br>
<br>
Copyright (C) 2004-2019 Institute of Molecular Function. All Rights Reserved.<br>
</font></p>
</td></tr><!--msnavigation--></table></body>

</html>
